feat(content): Tim the Stowaway - Human mission (#9977)
endless-sky/endless-sky: data/human/human missions.txt

			`	All of the hyped discussions around Hogsheads in the past gave them a brief but strong popularity amongst transport organizations looking to grab the average passenger's attention. In the end, the company responsible for this naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back, along with the interest of several pilots. Perhaps the name "Shippy McShipface" is fitting after all.`
				decline

mission "Timothy Radrickson 1: Stowaway"
	landing
	destination "Rand"
	source
		near "Zeta Aquilae" 3 10
		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
	to offer
		random < 10
	on enter

	# You will see this conversation after the first jump you make.
	on enter
		conversation

> 			`Upon completing your jump one of your internal sensors begins to beep. You flick off the siren and check the diagnostic report; apparently there is a non-negligable heat source detected in the food storage of your ship.`

Reviewer: ```suggestion
			`Upon completing your jump one of your internal sensors begins to beep. You flick off the siren and check the diagnostic report; apparently there is a non-negligible heat source detected in the food storage of your ship.`
```

			`	As problems go it is not a particularly alarming one; there are plenty of reasons there may be a heat spike in storage, and most of them would not require the captain of the ship to investigate.`
			`	It is unusual, however.`
			choice
				`	(Investigate.)`
					goto check
				`	(Ignore the sensor.)`

			action
				event "timothy radrickson: mummy" 41
			`	It's almost certainly nothing, and you have more important things to do than chase down every little sensor glitch on your ship.`
				decline

			label check
			`	After a short search, and the rearranging of quite a few deceptively heavy boxes, you are able to track the heat signature to a to be a foodstuff crate you recently took on. It has yet to be opened, but you can already tell its seal has been tampered with.`
			choice
				`	(Open the crate.)`
					goto crate
				`	(Dump the crate out the airlock.)`

> 			`	As the crate tumbles out into lifeless space your onboard sensors quickly note the anamolyous heat signature disappear. Whatever was in that crate is no longer your problem, and probably won't be anyone else's either.`

Reviewer: ```suggestion
			`	As the crate tumbles out into lifeless space your onboard sensors quickly note the anomalous heat signature disappear. Whatever was in that crate is no longer your problem, and probably won't be anyone else's either.`
```

		conversation
			`During the trip Timothy's spirits and health improved, and by the time you arrive on Rand he seems like a different man. Before he steps out of the airlock he turns to you one more time. "Thank you so much, Captain <last>." He shakes your hand."`
			`	"I mean it. From the bottom of my heart. You've saved my life, and given me hope. I won't forget this, or you. I can never thank you enough."`
			`	His eyes glistening, Timothy turns from you and exits your ship. Hopefully he finds what he is looking for on Rand.`
		event "timothy radrickson: on rand" 340

event "timothy radrickson: on rand"
event "timothy radrickson: mummy"

mission "Timothy Radrickson 2: mummy"
	landing
	to offer
		has "Timothy Radrickson 1: Stowaway: declined"
		has "event: timothy radrickson: mummy"
	on enter

	on enter
		conversation
			`For days there's been an odd lingering scent filtering in from your pressurized cargo; the smell has finally permeated your command deck, and something had to be done. A ship wide search has led you to a unopened foodstuffs crate. Although the crate was unopened, the seal on it appeared tampered with and the conjecture was that the food had gone bad.`

> 			`	Upon opening it you discovered that was in fact, not the case. Instead you discovered the decaying body of a dead and forgotten stowaway; the entire crate promptly found its way to your airlock. Despite the diligent work of your ship's airfilters, you can't quite rid the ship of the smell.`

Reviewer: ```suggestion
			`	Upon opening it you discovered that was in fact, not the case. Instead you discovered the decaying body of a dead and forgotten stowaway; the entire crate promptly found its way to your airlock. Despite the diligent work of your ship's air filters, you can't quite rid the ship of the smell.`
```

			`	"Where he's going there will be no bars, Tom," Ann quips. "They plan to send him to a work detail on Oblivion."`
			`	Tom gives that sort of news anchor laugh that's neither committal nor haughty. "In that case, forget what I said about 20 years, anything more than 5 years on Oblivion is a life sentence."`
			`	Both news anchors laugh, and then segue into a story about highly intelligent ferrets that have learned to sing in German.`
				accept

mission "Timothy Radrickson 5: Oblivion Meeting"
	landing
	to offer
		random < 25
		has "Timothy Radrickson 4: Imprisoned: active"
	source "Oblivion"
	deadline 1
	invisible
	on offer
		conversation
			`Your ship descends through the beautiful, toxic horizon and lands in the dirty, dismal spaceport outside of the provincial capital, Tukayyid. The clumped together warehouses they liberally call a spaceport are used as a central clearing house by all on-world industries, and workers from every industry filter in and out of this never-ending boom town. In the shadows, and on the fringes, the sick, and infirm linger, looking for a handout; the price of progress on such a hostile world.`

> 			`	Among the crowd one face draws your attention. A worker, unsupervized, is moving around a trolley of minerals by your ship as you have it refueled. His complexion is sallow, and his skin seems to droop off his bones. His blood red eyes, noticing your attention, languidly rise to look at you. Despite the damage, you realize you recognize him.`

Reviewer: ```suggestion
			`	Among the crowd one face draws your attention. A worker, unsupervised, is moving around a trolley of minerals by your ship as you have it refueled. His complexion is sallow, and his skin seems to droop off his bones. His blood red eyes, noticing your attention, languidly rise to look at you. Despite the damage, you realize you recognize him.`
```

			`	With that, he returns to moving the mineral trolley, leaving you to yourself.`
				accept

mission "Timothy Radrickson 5a: Timshank Redemption"
	name "Meet Tim's contact on <planet>"
	description "Tim would like you to collect some supplies for him, but first he asked you to meet someone on <stopovers> who could help you out. Tim has 18 days (<date>) before he is reassigned to a camp down south."
	cargo "Tim's equipment" 1
	deadline 18
	source Oblivion
	stopover Rand
	to complete
		"timothy radrickson medicine" >= 1
		"timothy radrickson suit" >= 1
		"timothy radrickson id" >= 1
	to offer
		has "Timothy Radrickson 5: Oblivion Meeting: active"
	on offer
		conversation
			`Even after donning a particulate mask, Oblivion's toxic atmosphere is so thick it seems as if you could swallow it; it doesn't help that the air somehow feels even denser at night. The warehouse, marked "Stephenson Heavy Industries," turns out to be at the very edge of the spaceport's boundary, and all you see when you get there is a lone box truck parked in the loading bay.`
			choice
				`	(Approach the truck.)`
				`	(Leave. Tim doesn't deserve another chance.)`
					goto leave

> 			`	By the truck you make out the silouette of a man heaving heavy equipment in the dim light of the loading bay. He turns to look at you as you come closer.`

Reviewer: ```suggestion
			`	By the truck you make out the silhouette of a man heaving heavy equipment in the dim light of the loading bay. He turns to look at you as you come closer.`
```

			`	As you walk away from the Stephenson warehouse you take a look at his list. Initially it's various survival items; backpacks, camping equipment and the like. There's also a specialized survival suit, and some heavy duty detoxification medication here as well. More alarmingly, however, are the industrial chemical compounds that would easily create a bomb, not to mention the gun at the bottom.`
			`	You're not familiar exactly what his crimes were, but you seem to recall them being mostly white-collar. Yet, you could potentially be arming a terrorist with these items. What might he blow up, who might he shoot, given his obvious desperation?`
			choice
				`	(Keep the list.)`
					goto keep
				`	(Throw away the list.)`

			`	It's not easy, walking away from the sad, shadow of a man that Timothy has become. But, you do it anyway. You toss the crumpled list into the trash, and leave Timothy to his fate.`
				decline

			label keep
			`	Timothy deserves another chance. You put the list back into your pocket and walk back to your ship.`
				accept

			label leave

> 			`	As you walk away, you hear the sound of a large garage door shuttering, the racket reverberating around the yard, followed shortly by the rumble of a truck's unhealthy engine reving up for a one way trip.`

Reviewer: ```suggestion
			`	As you walk away, you hear the sound of a large garage door shuttering, the racket reverberating around the yard, followed shortly by the rumble of a truck's unhealthy engine revving up for a one way trip.`
```

			choice
				`	(Leave your ship.)`
			`	It's dark and dusty outside. The planet's gravity makes you feel slow, and the air uncomfortably heavy in your lungs. The entrance to the mine looms large before you and seems suddenly quite menacing. Perhaps this wasn't the best idea...`
			`	"Don't move, or we will end you!" comes a shout from within the darkness of the mine. "Identify yourself, now!"`
			choice
				`	"I'm Captain <last>, veteran of the human civil war!"`
					goto timsbud
					to display
						has "free worlds plot completed"
				`	"I'm a friend of Timothy Radrickson!"`
					goto timsbud
				`	"I'm the Hero of Rand!"`
					goto timsbud
				`	"I'm just a merchant! My name's Captain <last>!"`
					goto timsbud
				`	"Mind your own business, I answer to no one!"`
				`	(Run back to your ship.)`
					goto "to the ship"

			`	You put these lowlife gangsters in their place. They're lucky you're not glassing their entire world, let alone deigning to speak with them. This is followed by a short moment of stunned silence, a snicker, and then the click of a trigger.`

> 			`	There is a brief, almost pleasant, moment of calm as the inevitablity of the situation sinks in... unfortunately, that is the last moment of calm you ever experience, as your body is riddled with bullet rounds mere seconds later.`

Reviewer: ```suggestion
			`	There is a brief, almost pleasant, moment of calm as the inevitability of the situation sinks in... unfortunately, that is the last moment of calm you ever experience, as your body is riddled with bullet rounds mere seconds later.`
```

			`	She tilts her head slightly. "But that's enough about me. What possible reason could you have for coming here?" Her face is open and friendly, but she casually rests her hand on her side arm.`
			choice
				`	(Tell her about Timothy.)`
					goto Timmy
				`	(Tell her you just wanted to say hi.)`
				`	(Tell her it is none of her business.)`

			`	She frowns and aims her gun squarely at your chest. "It's been too long since I've last seen you. And though you were once Governor Radrickson's friend, things have changed quite a bit since then."`
			`	She gives you a nonchalant shrug. "If that's all you have to say, then I am afraid there is no good reason to let you leave here alive." She favors you with a brief smile. "Operational security, you know how it goes."`
			`	She switches off the safety.`
			choice
				`	"Wait! Timothy sent me!"`
					goto sayso
				`	"Take your best shot."`
			`	She does indeed take her best shot, and the sudden hole in your torso proves to be quite terminal.`
				die

			label sayso
				`	She shakes her head. "Well why didn't you just say so! What does our Governor ask of us?" She holsters her sidearm.`

			label Timmy

> 			`	Union-Colonel Wynnfrith's eyes get misty as you tell her about Timothy's plight. She shakes her head a few times and then takes the list from you and carefully scrutinzes it.`

Reviewer: ```suggestion
			`	Union-Colonel Wynnfrith's eyes get misty as you tell her about Timothy's plight. She shakes her head a few times and then takes the list from you and carefully scrutinizes it.`
```

			`	"Ya know, to us he's still the Governor. Don't believe what the company-bosses and their paid politicians tell you. He was a good man." She wipes her eyes for a moment. "All he was trying to do was improve the conditions for workers here on Rand. He'd come up from the mines and he knew what it was like."`
			`	Her free hand tightens into a ball. "The big men didn't like that. But what about the profit margins? They tried to bribe him, to offer him anything he wanted, to just leave things well enough alone. To let the exploitation continue." She says the last words through clenched teeth.`

> 			`	"But he never waivered, and he always labored for the common man." Wynnafirth looks up at you. "You were a good friend to him, it's true, but truth be told, while you may have been his hero, he was ours; he was the real Hero of Rand.`

Reviewer: ```suggestion
			`	"But he never wavered, and he always labored for the common man." Wynnafirth looks up at you. "You were a good friend to him, it's true, but truth be told, while you may have been his hero, he was ours; he was the real Hero of Rand.`
```

			`	"And when he wouldn't budge they finally got him for trumped up charges and threw him to Oblivion to disappear and die." She shakes her head. "There was nothing we could do. We're still in a..." she hesitates, "... a rebuild phase."`
			`	She reads the list again. "A few weeks, you say? Well, we already thought he was dead, so if there's a chance we save the Governor then we're in.`

> 			`	"We have a few left over exposives and detonators from when this mine was active. That should take care of the bomb he was hoping for." She then unholsters her sidearm and hands it to you.`

Reviewer: ```suggestion
			`	"We have a few left over explosives and detonators from when this mine was active. That should take care of the bomb he was hoping for." She then unholsters her sidearm and hands it to you.`
```

			choice
				`	(Take it.)`

			`	"And this should take care of his need for a gun. As for the other items on there..." She turns to a torn and faded star chart taped on the rocky cavern wall.`

> 			`	Grabbing a small stick leaning from the wall, she points to Rastaban. "Dancer would be the best world to find a survival suit rated for Oblivion's toxic environment. As for medication..." She looks over the chart for a moment, then points to Dabih. "Your best bet will be New Britain. One of the few worlds in the Belt that can produce complex pharmecuticals."`

Reviewer: ```suggestion
			`	Grabbing a small stick leaning from the wall, she points to Rastaban. "Dancer would be the best world to find a survival suit rated for Oblivion's toxic environment. As for medication..." She looks over the chart for a moment, then points to Dabih. "Your best bet will be New Britain. One of the few worlds in the Belt that can produce complex pharmaceuticals."`
```

> 			`	The pointing stick falls on Shaula next. "The best place for a fake ID would be the 'independent' world of Greenrock." She taps her chin before continuing, "It would be tight though getting to all three of these worlds and back to the governor in time. Any of these worlds may also have the other equipiment you need, but the quality likely wouldn't be as good."`

Reviewer: ```suggestion
			`	The pointing stick falls on Shaula next. "The best place for a fake ID would be the 'independent' world of Greenrock." She taps her chin before continuing, "It would be tight though getting to all three of these worlds and back to the governor in time. Any of these worlds may also have the other equipment you need, but the quality likely wouldn't be as good."`
```

			`	She turns back to you. "I am sorry we couldn't be more help, but hopefully it's enough."`
			`	After putting the pointing stick back in its place, Wynnafrith steps through the door, indicating for you to follow her.`
			choice
				`	(Follow her.)`

> 			`	Once again in the largern cavern she turns and points at the guard. "Grab our explosive and carefully accompany Captain <last> with it back to his ship."`

Reviewer: ```suggestion
			`	Once again in the larger cavern she turns and points at the guard. "Grab our explosive and carefully accompany Captain <last> with it back to his ship."`
```

	landing
	source Rand
	destination "New Britain"
	to offer
		has "Timothy Radrickson 5a: Timshank Redemption: active"
	on complete
		conversation
			`With a little effort you're able to navigate New Britain's labyrinthian ziggurat masquerading as a spaceport. The lower levels are dedicated to a bustling trade sector where merchants, traders, and producers from all over the Dirt Belt haggle to sell their raw materials, and purchase complex equipment and refined products.`
			label options
			choice
				`	(Find the detoxification medicine.)`
					goto med1
					to display
						not "med1"
				`	(Haggle with the medicine salesman again.)`
					goto med2
					to display
						has "med1"
						not "med2"
				`	(Look for the survival suit.)`
					goto suit1
					to display
						"timothy radrickson suit" <= 1
						not "suit1"
				`	(Return to the shop to buy the survival suit.)`
					goto suit2
					to display
						has "suit1"
						not "suit2"
				`	(Find someone that can procure a fake ID for you.)`
					goto id1
					to display
						"timothy radrickson id" = 0
						not "id1"
				`	(Leave the trade sector.)`
					goto end

			label med1
			action
				set "med1"

> 			`	You make your way to the pharmecutical wing, through crowds of representatives from large conglomorates in the process of wrangling with representatives from various worlds, all needing exotic medicine for their innumerable plagues, blights, pandemics, and the like. There's a booth with a slick salesman selling expensive drugs promising to reverse and protect from the effect of toxic air and radiation, and it just so happens that Oblivion is their demonstration planet.`

Reviewer: ```suggestion
			`	You make your way to the pharmaceutical wing, through crowds of representatives from large conglomerates in the process of wrangling with representatives from various worlds, all needing exotic medicine for their innumerable plagues, blights, pandemics, and the like. There's a booth with a slick salesman selling expensive drugs promising to reverse and protect from the effect of toxic air and radiation, and it just so happens that Oblivion is their demonstration planet.`
```

			`	"There's no need to fear Oblivion's deleterious effects so long as you take one of these a day!" the salesman says brightly. "Why, you could walk among your condemned workers unmasked, and demonstrate your superior executive prowess!"`
			`	You listen on to the salesman, who reckons he can cut you in on a special deal for 140,000 credits.`
			choice
				`	(Purchase the medicine for Tim.)`
					goto "med 140"
				`	(Haggle for a lower price.)`
					goto haggle
				`	(Pass.)`
			`	That price is too steep. You return to the central atrium of the trade sector to consider your options.`
				goto options
			
			label haggle

> 				`The salesman's eyes narrow and his face suddenly looks vaguely reptillian. "I'm already giving you the best price. The literal shirt off my back!"`

Reviewer: ```suggestion
				`The salesman's eyes narrow and his face suddenly looks vaguely reptilian. "I'm already giving you the best price. The literal shirt off my back!"`
```

			choice
				`	(Offer 60,000 for the meds.)`
				`	(Offer 120,000 for the meds.)`
					goto justright
				`	(Agree to pay 140,000 for the meds.)`
					goto "med 140"
				

> 			`	The drug saleman's eyes flash with hostility before he breaks out in an angry laugh. "Who do you think you are? What do you think, this is a charity? I already have a buyer lined up for more than twice that!"`

Reviewer: ```suggestion
			`	The drug salesman's eyes flash with hostility before he breaks out in an angry laugh. "Who do you think you are? What do you think, this is a charity? I already have a buyer lined up for more than twice that!"`
```

			`	He begins packing up his equipment. "I'm charging you full price, 140,000 or nothing. Last chance."`
			choice

> 				`	(Purchase the medcine for Tim.)`

Reviewer: ```suggestion
				`	(Purchase the medicine for Tim.)`
```

					goto "med 140"
				`	(Pass.)`
			action
				set "med2"

> 			`	That price is too expensive; you'll have to see if you can purchase a less potent and cheaper drug elsewhere. You return to the central atrium of the trade sector to consider your other options.`

Reviewer: ```suggestion
			`	That price is too high; you'll have to see if you can purchase a less potent and cheaper drug elsewhere. You return to the central atrium of the trade sector to consider your other options.`
```

				goto options

			label justright

> 				`	The saleman glowers, but nods his head slightly. "Fine! 120,000 credits it is. My family will starve, but at least you'll get your precious drugs." He begins packing them up for you.`

Reviewer: ```suggestion
				`	The salesman glowers, but nods his head slightly. "Fine! 120,000 credits it is. My family will starve, but at least you'll get your precious drugs." He begins packing them up for you.`
```

					goto "med 120"

			label med2
			action
				set "med2"
			`	You decide to go back the pharmaceutical wing to reconsider the deal.`
			`	"Here's the captain again!" says the salesman, as if he'd known you for years. He leans in uncomfortably close to you and says, "I can tell you're a discerning customer, so here's what I will let these go for: 120,000 credits. That's almost wholesale price! I'm barely making a cent, but I really need to unload all these drugs."`
			`	He favors you with an unnaturally white, toothy grin.`
			choice
				`	(Purchase the discounted medicine for Tim.)`
					goto "med 120"
				`	(Haggle for an even lower price.)`
					goto haggle2
				`	(Pass.)`
			`	Again, the medicine just costs too much; Tim will have to survive with something less potent.`
			`	The salesman's eyes bulge, and his smooth face is suddenly crisscrossed on unattractive wrinkles. "Fine. I have another buyer anyways," he half-growls at you. He begins packing up his stall, muttering and shaking his head to himself as he does so.`
			`	You return to the central atrium of the trade sector to consider your other options.`
				goto options
			
			label haggle2

> 			`	The drug saleman's eyes flash with hostility before he breaks out in an angry laugh. "Who do you think you are? What do you think, this is a charity? I already have a buyer lined up for more than twice what I've offered you!"`

Reviewer: ```suggestion
			`	The drug salesman's eyes flash with hostility before he breaks out in an angry laugh. "Who do you think you are? What do you think, this is a charity? I already have a buyer lined up for more than twice what I've offered you!"`
```

			`	The trade sector has all manner of ID services and consultants, but you find none of the respectable ones are willing to talk to you after you explain the ID has to be prepared "in absentia," and the unrespectable ones appear to be lacking in this spaceport given the presence of the spaceport gendarme at every corner. Frustrated, you decide to take a walk outside the spaceport proper.`
			choice
				`	(Give up and go get something else.)`
					goto options
				`	(Keep walking about.)`
			`	You eventually reach a little forested garden area with a few benches, by one of the parking lots that serve the spaceport. The suns of Dabih are out in full force today, and you're glad to be in the shade.`
			`	As you recover from the heat you notice that, sitting over in the parking lot, there's a few teenagers chatting amongst themselves. The sort that look like they'd know where to get a false ID if they needed one. Perhaps they could give you some pointers.`
			choice
				`	(Give up and go get something else.)`
					goto options
				`	(Approach the teenagers for help about the ID.)`
			`	The teenagers, two of them guys and one a girl, grin at you like fresh meat as you near them.`

> 			`	The guy in the middle says, "What'dya need, boss? Wanna get high? Wanna get low? Wanna get out?" Apparently this group is used to working with frustrated castouts from the overly legal spaceport emporium.`

Reviewer: ```suggestion
			`	The guy in the middle says, "What'dya need, boss? Wanna get high? Wanna get low? Wanna get out?" Apparently this group is used to working with frustrated outcasts from the overly legal spaceport emporium.`
```
Is this what you mean?

Author: I meant cast outs

	landing
	source Rand
	destination Dancer
	to offer
		has "Timothy Radrickson 5a: Timshank Redemption: active"
	on complete
		conversation
			`Your ship shakes as it descends into Dancer's mad weather. Lightning crackles beyond your sturdy bulkheads and a panoply of colorful clouds rushes past as you boost down into in the dull overcast light of a subdued day on Dancer.`
			`	Around the quiet spaceport there's a smattering of low reinforced buildings linked together by weather worn trails dug into the ground with protective earth on either side. You see one leading to a spacious, if squat, clearing house.`
			choice
				`	(Enter the clearing house.)`
			`	There is an eclectic assortment of gear, goods, supplies and equipment here. Some new, some used to the point of ruin. Unsurprisingly, given the primary industry of this world, much of it is scientific.`
			`	A middle aged woman with deep windburned scars and a striking shock of white between her otherwise dark hair greets you as you come in. "Anything worth having on Dancer can be found here. Just take a look around and let Bertha know if you find anything to your liking."`
			label danceroptions
			choice

> 				`	(Look at the survial suits.)`

Reviewer: ```suggestion
				`	(Look at the survival suits.)`
```

					goto suit1
					to display
						not "suit1"
				`	(Look at the survival suits again.)`
					goto suit2
					to display
						has "suit1"
						not "suit2"
				`	(Check the medicine shelves.)`
					goto med1
					to display
						"timothy radrickson medicine" = 0
						not "med1"
				`	(Return to the medicine shelves.)`
					goto med2
					to display
						has "med1"
						not "med2"
				`	(Rummage around for old IDs.)`
					goto id1
					to display
						"timothy radrickson id" <= 1
						not "id1"
				`	(Go back to where you found the ID.)`
					goto id2
					to display
						has "id1"
						not "id2"
				`	(Leave the clearing house.)`
					goto end

			label suit1
			action
				set "suit1"

> 			`	There are a handful of surrival suits on display here. These are durable, hand-crafted suits carefully checked and rated to ensure the highest chance of survival against the fierce uncaring winds of Dancer.`

Reviewer: ```suggestion
			`	There are a handful of survival suits on display here. These are durable, hand-crafted suits carefully checked and rated to ensure the highest chance of survival against the fierce uncaring winds of Dancer.`
```

> 			`	Old Bertha notices your attention and walks up to you. "Going anywhere overland on Dancer without one of these beauties is a deathwish."`

Reviewer: ```suggestion
			`	Old Bertha notices your attention and walks up to you. "Going anywhere overland on Dancer without one of these beauties is a death wish."`
```

			`	She winks at you, as if she knows something you don't. "Worry not though, hun. We only sell the highest rated and best-tested suits here. I won't have any one's death on my conscience, 'sides, it's bad for business."`
			`	She pinches some of the dense microwoven fabric of the nearest suit. "It's not cheap, but neither is your life. I'll sell ya one of these for 300,000 credits."`
			choice
				`	(Purchase a survival suit for Tim.)`
					goto getsuit
				`	(Haggle to try and lower the price.)`
					goto haggle
				`	(Pass.)`
			`	300,000 credits is way too much. The woman shrugs and steps away as you turn your attention back to the rest of the store.`
				goto danceroptions
			
			label haggle
				`	Bertha gives you a sad smile and shakes her head slowly. "No room here for haggling, hun. These are the best of the best and they are worth every red cent. I am sure you can find bottom bin ones elsewhere, but here you pay for quality."`
				`	She folds her arms. "Now, it's 300,000 credits or nothing."`
			choice
				`	(Purchase the survival suit for Tim.)`
					goto getsuit
				`	(Pass.)`

> 			`	Bertha sighs and then shrugs, and leaves you alone to continue browsing through her clearning house.`

Reviewer: ```suggestion
			`	Bertha sighs and then shrugs, and leaves you alone to continue browsing through her clearing house.`
```

				goto danceroptions
				
			label suit2
			action
				set "suit2"
			`	You loop back to where the well-crafted survial suits hang. The price is still 300,000 credits.`
			choice
				`	(Purchase a survival suit for Tim.)`
					goto getsuit
				`	(Go back to browsing.)`
			`	Not ready to make a purchase, you return to perusing the store.`
				goto danceroptions

			label getsuit
			action
				set "suit2"
				"timothy radrickson suit" = 3
				payment -300000
			`	Bertha is happy to ring you up, and gives you the carefully packaged and sealed survival suit for <payment>.`
				goto danceroptions
		
			label med1
			action
				set "med1"
			`	There's numerous shelves filled with all kinds of drugs, and it takes you a few minutes to find the detoxification medicine. By then Bertha is at your side.`

> 			`	"Foul winds are known to kick up toxic dust in the atomosphere, and if you plan on staying here long you're likely going to need to take a few of these." She looks at you curiously, and then adds, "However, you may find them less useful on other worlds."`

Reviewer: ```suggestion
			`	"Foul winds are known to kick up toxic dust in the atmosphere, and if you plan on staying here long you're likely going to need to take a few of these." She looks at you curiously, and then adds, "However, you may find them less useful on other worlds."`
```

			`	Finally you reach the center of the emporium and discover a crudely drawn map of the emporium. It's not much, but at least it tells you where you can find what you're looking for.`
			label greenoptions
			choice
				`	(Find someone that can produce a fake ID for you.)`
					goto id1
					to display
						not "id1"
				`	(Go back to Ivaldur's forging stall for the fake ID.)`
					goto id2
					to display
						has "id1"
						not "id2"
				`	(Look for a survival suit.)`
					goto suit1
					to display
						"timothy radrickson suit" = 0
						not "suit1"
				`	(Return to the shop to buy the survival suit.)`
					goto suit2
					to display
						has "suit1"
						not "suit2"
				`	(Find the detoxification medicine.)`
					goto med1
					to display
						"timothy radrickson id" <= 1
						not "id1"
				`	(Go back to the drug dealer to get the detoxification medicine.)`
					goto med2
					to display
						has "med1"
						not "med2"
				`	(Leave the emporium.)`
					goto end
			
			label id1
			action
				set "id1"

> 			`	There's more than a half dozen stalls all advertising their high skills and production value when it comes to the art of crafting fake IDs. Most the stalls and their employees have a garish, sloppy, and boostful manner, but one with a more professional and constrained feel attracts your attention; it's run by an older gentleman who has the look of a respectable watchmaker.`

Reviewer: ```suggestion
			`	There's more than a half dozen stalls all advertising their high skills and production value when it comes to the art of crafting fake IDs. Most the stalls and their employees have a garish, sloppy, and boastful manner, but one with a more professional and constrained feel attracts your attention; it's run by an older gentleman who has the look of a respectable watchmaker.`
```

			`	He nods at you politely as you approach. "Good day to you. I am Ivaldur the Forger, and what can I do for you?"`
			choice
				`	(Ask about producing a fake ID.)`
					goto ask
				`	(Turn around and leave.)`
			`	For whatever reason you are not interested in doing business with Ivaldur the Forger, and promptly head back out into the emporium.`
				goto greenoptions
			
			label ask

> 			`	The old man smiles at you good naturedly. "Simple enough request. I can provide you with an ID, and a complex matching background that will pass all but the most exhasustive of checks."`

Reviewer: ```suggestion
			`	The old man smiles at you good-naturedly. "Simple enough request. I can provide you with an ID, and a complex matching background that will pass all but the most exhaustive of checks."`
```

				set "suit2"
				"timothy radrickson suit" = 1
				payment -250000
			`	The proprietor quickly rings you up as if worried you'll change your mind at any moment, and gleefully takes your <payment>.`
			`	"Safe travels, privateer," he says as you leave his stall.`
				goto greenoptions

			label med1
			action
				set "med1"
			`	There are many stalls loudly advertising all kinds of illegal drugs, and it takes some work to sift through them until you come across one selling legal drugs as well. The man running it is sallow and pock-marked, but his eyes are keen. He almost hisses as he greets you.`
			`	"Whatever you need, wherever you're going. I have you covered."`
			choice
				`	(Ask about detoxification medicine for Oblivion.)`
					goto askdrugs
				`	(Leave.)`
			`	This is not the place for you. You take your leave and return to the emporium at large.`
				goto greenoptions

			label askdrugs
			`	The drug-dealer flashes you with his yellow teeth and unnerving smile. "For every kind of toxic environment under any sun." He jolts towards his little wall of drawers, pulling out one labeled "Oblivion detoxifiers." From the drawer he picks a bottle up, checking its label.`

> 			`	"It says it expired a few years ago, but that's just so the corpo's can squeeze the common man, ya know? Stuff's still good." He shrugs.`

Reviewer: ```suggestion
			`	"It says it expired a few years ago, but that's just so the corpos can squeeze the common man, ya know? Stuff's still good." He shrugs.`
```

			`	"You did it?" His words more strained than the last time you spoke. "You got the gear?"`
			`	He looks down at your dolly expectantly.`
			choice
				`	"It's all here."`
				`	"Of course I did."`
				`	"Yep, but you owe me big."`
			`	He nearly drops to his knees and starts sorting through the gear. He takes heart when you tell him about Wynnafirth's gun and bomb, and how highly she still thinks of him.`
			`	"Good secretary. Better friend." He wheezes as he checks out the rest of your gear.`
			branch unprepared
				"timothy radrickson preparedness" < 6

			branch "decently prepared"
				"timothy radrickson preparedness" >= 6
				"timothy radrickson preparedness" <= 7

			branch "well prepared"
				"timothy radrickson preparedness" > 7

			label unprepared
			`	As he inspects the suit, drugs, and ID you can see his demeanor shift. He seems to inflate a little.`
			`	Finally, he straightens back up. "You did the best you could. I'm sure." He eyes the gear wearily. "At least it's a chance."`
				goto end

			label "decently prepared"

> 			`	As he inspects the suit, drugs, and ID you he nods to himself satisfactorialy.`

Reviewer: ```suggestion
			`	As he inspects the suit, drugs, and ID you he nods to himself satisfactorily.`
```

		has "event: timothy radrickson forever"
		or
			"timothy radrickson preparedness" < 6
			has "Timothy Radrickson 5a: Timshank Redemption: failed"
	on offer
		conversation
			`Oblivion's spaceport is the kind of place that, despite being in constant motion, never really changes. Humanity, driven by its constant thirst for resources, scurries over this unremarkable world like a hive of angry bees, but for all their furious effort it looks no different than any of the previous times you've been here.`
			`	Except, of course, the lack of one familiar face. A doomed and disgraced ex-governor who had been sent here, and last you heard, had died here.`
			`	With that heavy thought on your mind you notice a sign which promises to lead you to the local graveyard.`
			choice
				`	(Head to the graveyard.)`
					goto graveyard
				`	(Head back to your ship.)`
			`	You see no point in wasting your time at this place's graveyard, so you head back to your ship.`
				decline
			
			label graveyard
			`	Oblivion's evening sky is lit up by unnumbered rigs and natural gas burners that dot the region. These red fires turn the normally dark blue atmosphere a shade of smoldering purple.`

> 			`	It's beneath that purple hue you see a countless expanse of small flat gravestones uniformally placed in Oblivion's sterile dirt. Despite the number of headstones there is no one else here, and only a few graves show any evidence of being visited.`

Reviewer: ```suggestion
			`	It's beneath that purple hue you see a countless expanse of small flat gravestones uniformly placed in Oblivion's sterile dirt. Despite the number of headstones there is no one else here, and only a few graves show any evidence of being visited.`
```

			`	It seems as though those buried here are mostly forgotten.`
			choice
				`	(Search for Timothy Radrickson's grave.)`
					goto search
				`	(Leave.)`
			`	You've seen all you wanted to see of this place, and leave this unloved graveyard to its forlorn dead.`
				decline

			label search
			`	The index is easy to find, and has all the hallmarks of having never been used. The graveyard is set up in a carefully regimented grid system and despite the large number of graves, it is surprisingly easy finding Timothy's.`

> 			`	You walk across the forgotten memorial to find a small, flat unremarkable stone, hemmed in amongst other unremarkable stones, pressed into the dead ground. The text laser-chisled onto it reads, "Timothy Radrickson. Prisoner - 645631-4C"`

Reviewer: ```suggestion
			`	You walk across the forgotten memorial to find a small, flat unremarkable stone, hemmed in amongst other unremarkable stones, pressed into the dead ground. The text laser-chiseled onto it reads, "Timothy Radrickson. Prisoner - 645631-4C"`
```

			`	That's all. Nothing about his meteoric rise, about being the Governor of Rand, or even how or when his death occurred here. Just another dead prisoner.`
			choice
				`	(Stay a while and contemplate.)`
				`	(Leave.)`
					goto end

> 			`	The slow moving evening clouds have cast the entire world in a strange purple twilight. Timothy's grave sits before you, nestled in the poision ground, and you alone in the vast empty graveyard.`

Reviewer: ```suggestion
			`	The slow moving evening clouds have cast the entire world in a strange purple twilight. Timothy's grave sits before you, nestled in the poison ground, and you alone in the vast empty graveyard.`
```

			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh while others seem disappointed or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - Shippy McShipface. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
			`	All of the hyped discussions around Hogsheads in the past gave them a brief but strong popularity amongst transport organizations looking to grab the average passenger's attention. In the end, the company responsible for this naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back, along with the interest of several pilots. Perhaps the name "Shippy McShipface" is fitting after all.`
				decline

mission "Timothy Radrickson 1: Stowaway"
	invisible
	landing
	source
		near "Zeta Aquilae" 3 10

> 		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"

Reviewer 2: Some none required quotes could be removed.

> 		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"

Reviewer 2: Some quotes could be removed here.

Author: This might be something that has changed in the two years between when I wrote this part of the mission, and now. But I don't actually know what quotes need be removed here, or how it should look without them.

Reviewer 3: The quotes aren't necessary, but for gov'ts they seem to be all in quotation marks in the data files.

	destination "Rand"
	to offer
		random < 10
	on enter

	# You will see this conversation after the first jump you make.
	on enter
		conversation

> 			`Upon completing your jump one of your internal sensors begins to beep. You flick off the siren and check the diagnostic report; apparently there is a non-negligible heat source detected in the food storage of your ship.`

Reviewer 2: ```suggestion
			`Upon completing your jump, one of your internal sensors begins to beep. You flick off the siren and check the diagnostic report: apparently there is a non-negligible heat source detected in the food storage of your ship.`
```

> 			`	As problems go it is not a particularly alarming one; there are plenty of reasons there may be a heat spike in storage, and most of them would not require the captain of the ship to investigate.`

Reviewer 2: ```suggestion
			`	As problems go, it is not a particularly alarming one: there are plenty of reasons there may be a heat spike in storage, and most of them would not require the captain of the ship to investigate.`
```

			`	It is unusual, however.`
			choice
				`	(Investigate.)`
					goto check
				`	(Ignore the sensor.)`

			action
				fail
				event "timothy radrickson: mummy" 41

> 			`	It's almost certainly nothing, and you have more important things to do than chase down every little sensor glitch on your ship.`

Reviewer 2: ```suggestion
			`	It's almost certainly nothing, and you have more important tasks to do than chase down every little sensor glitch on your ship.`
```

Author: I am on the fence about this one. While tasks sounds more professional than things, "_more important things to do_" is in of itself a well used phrase, so it feels natural when I read it. Whereas "tasks to do" feels unnatural to me. Like it's being forced.

Reviewer 2: It doesn't for me but, that's just personal preference. If you fell more about "things" than "tasks", then leave it as it is, it's already good like that.

			label check
			`	After a short search, and the rearranging of quite a few deceptively heavy boxes, you are able to track the heat signature to a foodstuff crate you recently took on. It has yet to be opened, but you can already tell its seal has been tampered with.`
			choice
				`	(Open the crate.)`
					goto crate
				`	(Dump the crate out the airlock.)`

			action
				fail
			`	As the crate tumbles out into lifeless space your onboard sensors quickly note the anomalous heat signature disappear. Whatever was in that crate is no longer your problem, and probably won't be anyone else's either.`

			label crate

> 			`	You pry open the crate. It should be full of meals for you and your crew, but after removing the lid you discover a dirty and disheveled man.`

Reviewer 2: ```suggestion
			`	You pry open the crate. It should be full of meals for you and your crew, but after removing the lid you discover a dirty disheveled man.`
```

Note sure about "pry"...

> 			`	Before you have a chance to say anything the man looks up at you with narrowed, pained eyes, and whimpers. "P-p-please. I just had to get away. I-I-It wasn't safe. I had to get away." He weakly holds up his hands.`

Reviewer 2: ```suggestion
			`	Before you have a chance to say anything, the man looks up at you with narrowed, pained eyes, and whimpers. "P-p-please. I just had to get away. I-I-It wasn't safe. I had to get away." He weakly holds up his hands.`
```

			choice
				`	"Who are you?"`
				`	"Where's my food?"`

> 			`	T-T-Timothy Radrickson. I-I tossed your food out to make room. I am sorry. I am so sorry. I have no money to pay you back. I just h-h-had to leave. They were after me. I owned money. I had nothing. Please. Any other Republic world will do. I just need to get away. I just need to start anew. I'll p-p-pay you back I swear. I swear. I will. I swear.`

Reviewer 2: ```suggestion
			`	"T-T-Timothy Radrickson. I-I tossed your food out to make room. I am sorry. I am so sorry. I have no money to pay you back. I just h-h-had to leave. They were after me. I owed money. I had nothing. Please. Any other Republic world will do. I just need to get away. I just need to start anew. I'll p-p-pay you back I swear. I swear. I will. I swear."`
```

			choice
				`	"Fine. I'll drop you off at the next port."`
					goto port
				`	"Is there somewhere you'd like to go?"`
					goto somewhere

> 				`	"I know where you're going. The airlock."`

Reviewer 2: Ahah, that's cruel.

			action
				fail

> 			`	The feeble man screams as you drag him out of the crate and toward the airlock. He ineptly squirms as you shove him pass the bulkhead seals. He falls to his knees, shudders with terror, and his face leaks tears. You close the airlock, silencing his shrill screams, and hit the button to open it on the outside. The man is promptly sucked out into cold, cruel space and crystallizes before becoming this system's newest comet.`

Reviewer 2: ```suggestion
			`	The feeble man screams as you drag him out of the crate and toward the airlock. He ineptly squirms as you shove him pass the bulkhead seals. He falls to his knees, shudders with terror, and his face leaks tears. You close the airlock, silencing his shrill screams, and hit the button to open the outside airlock. The man is promptly sucked out into cold, cruel space and crystallizes before becoming this system's newest comet.`
```

"outside airlock" is just personal preference.

			label port
			action
				fail
				set "timothy radrickson: port"

> 			`	Tears squeeze out the man's sad eyes. "Thank you," he whispers, "thank you."`

Reviewer 2: ```suggestion
			`	Tears squeeze out the man's sad eyes. "Thank you", he whispers, "thank you."`
```

> 			`	You get the man cleaned up and settled and head back to your station.`

Reviewer 2: ```suggestion
			`	You get the man cleaned up and settled and head back to the station.`
```
Not sure about that.

Author: maybe... "You get the man cleaned up and settled, and then go back to running your ship"?

or.."captaining your ship."?
or.. "piloting your ship."?

Reviewer 2: I like "You get the man cleaned up and settled, and then go back to running your ship"

Author: Made the change!

			label somewhere

> 			`	Timothy looks at you, his eyes wide and full of tears. "Rand," he whispers. "Rand. I can make good money there. I can turn my life around. B-b-but I'd be okay with anywhere. You d-d-don't have to.."`

Reviewer 2: ```suggestion
			`	Timothy looks at you, his eyes wide and full of tears. "Rand", he whispers. "Rand. I can make good money there. I can turn my life around. B-b-but I'd be okay with anywhere. You d-d-don't have to.."`
```

Reviewer 3: This isn't correct. The comma goes within the quotation mark. Same for the above suggestion.

			`	You cut him off. "It's fine, Timothy. I'll take you to Rand."`
			`	Timothy breaks down in tears. "Thank you. Thank you. Thank you..." he mumbles over and over again.`
			`	You help him out of the crate and get him cleaned up.`
				accept

	on complete
		event "timothy radrickson: on rand" 340
		conversation

> 			`During the trip Timothy's spirits and health improved, and by the time you arrive on Rand he seems like a different man. Before he steps out of the airlock he turns to you one more time. "Thank you so much, Captain <last>." He shakes your hand."`

Reviewer 2: ```suggestion
			`During the trip Timothy's spirits and health improved, and by the time you arrive on Rand he seems like a different man. Before he steps out of the airlock he turns to you one more time. "Thank you so much, Captain <last>." He gives you a handshake."`
```
Just personal preference.

Author: I like the more direct way it's currently written.

> 			`	"I mean it. From the bottom of my heart. You've saved my life, and given me hope. I won't forget this, or you. I can never thank you enough."`

Reviewer 2: ```suggestion
			`	"I mean it, from the bottom of my heart. You've saved my life, and given me hope. I won't forget this; or you. I could never thank you enough."`
```

			`	His eyes glistening, Timothy turns from you and exits your ship. Hopefully he finds what he is looking for on Rand.`

mission "Timothy Radrickson 1a: Accepted"
	landing
	name "Stowaway to <planet>"

> 	description "You found Timothy on your ship stowed away in a food crate, on the run from someone. He requests you to bring him to <destination>, where he can find work."

Reviewer 2: ```suggestion
	description "You have found Timothy on your ship stowed away in a foodstuff crate, on the run from somebody. He requests you to bring him to <destination>, where he can find work."
```
Just personal preference.

	source

> 		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"

Reviewer 2: Some unnecessary quotes again.

	destination "Rand"
	to offer
		has "Timothy Radrickson 1: Stowaway: active"
	on abort
		fail "Timothy Radrickson 1: Stowaway"

event "timothy radrickson: on rand"
event "timothy radrickson: mummy"

mission "Timothy Radrickson 2: mummy"
	landing
	invisible
	to offer
		has "Timothy Radrickson 1: Stowaway: failed"
		has "event: timothy radrickson: mummy"

> 	on enter

Reviewer 2: It's duplicated.

> 	on enter

Reviewer 2: Still duplicated here.

Author: That's on purpose. Duplicated on enter means that it should happen when you are in space after a jump, and not on planet. 
So you notice your ship stinking of dead-dude in space and not on world.

Reviewer 2: Okay, you've learned me something.

		conversation

> 			`For days there's been an odd lingering scent filtering in from your pressurized cargo; the smell has finally permeated your command deck, and something had to be done. A ship wide search has led you to a unopened foodstuffs crate. Although the crate was unopened, the seal on it appeared tampered with and the conjecture was that the food had gone bad.`

Reviewer 2: ```suggestion
			`For days there's been an odd lingering scent filtering in from your pressurized cargo. The smell has finally permeated your command deck, and something had to be done. A ship-wide search has led you to a unopened foodstuff crate. Although the crate was unopened, the seal on it appeared tampered with and the conjecture was that the food had gone bad.`
```

Not sure about "ship-wide"

			`	Upon opening it you discovered that was in fact, not the case. Instead you discovered the decaying body of a dead and forgotten stowaway; the entire crate promptly found its way to your airlock. Despite the diligent work of your ship's air filters, you can't quite rid the ship of the smell.`

mission "Timothy Radrickson 2: Interlude"
	landing
	to offer
		has "Timothy Radrickson 1: Stowaway: failed"
		has "timothy radrickson: port"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	on offer
		conversation

> 			`Blinking at the light Timothy shuffles toward the exit. "Th-th-thank you. Thank you so much," Timothy mumbles.`

Reviewer 2: ```suggestion
			`Blinking at the light, Timothy shuffles toward the exit. "Th-th-thank you. Thank you so much," Timothy mumbles.`
```

			`	Timothy opens the hatch and looks back at you. "You s-s-saved my life. I won't forget. I won't."`

> 			`	With that Timothy exits your ship and begins his new life. You hope everything works out for the best.`

Reviewer 2: ```suggestion
			`	With that, Timothy exits your ship and begins his new life. You hope everything will work out for the best.`
```

Author: good with the added comma, thor knows my comma usage is terrible, but adding 'will' just comes off to me as slightly more wordy for the sake of being slightly more wordy?

Reviewer 2: I agree after looking back at it.

		conversation
			`After your ship touches down you notice a light blinking on your dashboard. You have a cached message waiting for you to play.`
			choice
				`	(Play the message.)`

			`	The screen lights up with the familiar, fuller face of a healthy Timothy Radrickson.`
			`	"Captain <last>. Let me just reiterate my thanks to you for what you did for me. Most captains would have airlocked me, or dropped me off wherever, but you really gave me a chance."`
			`	He stops and grins at the camera for a moment. "And that chance, it really has made all the difference. It's been a year of hard work, but I've saved some money and made some investments, and things really are turning around."`
			`	"As way of thanks," he continues, "I've included 50,000 credits with this message. That should cover the cost of transporting to me to Rand, but it doesn't come to close to what I really owe. Which is, in all honesty, my very life."`
			`	He pauses, as if he's trying to remember something. "Anyways, thank you again, and know I'll never forget what you did for me."`
			`	Timothy waves and then the video ends.`
				decline

event "timothy radrickson: governor"

> # Plan:

Reviewer 2: Nice idea!

# Tim invites the PC to come and visit him on Rand. He takes you to his governor palace, thanks you for all you did, gives you 125,000 for giving him a ride all those years ago.
# He talks about how Rand just sent out the largest mineral freighter fleet ever to New Britain, and when it returns the colony will finally have enough money to start taking care of it self...
# When.. uh oh the frieghters have lost communication or sent out a distress signal or something, and you offer to help.
# You fly off, find a marauder/pirate fleet, destroy them, save the freighter fleet, go back to Rand and then Tim holds a swanky dinner and names you Hero of Rand.

mission "Timothy Radrickson 3: Governor Notification"
	name "Visit the Governor on <planet>"

> 	description "The Governor of <planet> has invited you to speak with them."

Reviewer 2: ```suggestion
	description "The governor of <planet> has invited you to speak with them."
```
I don't think the cap's required.

Reviewer 3: You're referring to a specific governor here, not just governors in general. I believe that is a criterion for capitalising it.

	to offer
		has "event: timothy radrickson: governor"
	destination "Rand"
	landing
	on offer
		conversation
			`An indicator light is blinking on your main panel. There is a cached message waiting for you to play.`
			choice
				`	(Play the message.)`

> 			`	A young woman in very professional clothing appears on the screen. "Captain <last>, I am pleased to inform you that the Governor of Rand requests your presence. Please come to Rand at your earliest convenience.`

Reviewer 2: ```suggestion
			`	A young woman in very professional clothing appears on the screen. "Captain <last>, I am pleased to inform you that the governor of Rand requests your presence. Please come to Rand at your earliest convenience.`
```

I don't think the cap's required.

				`	"Where's my food?"`

			`	"T-T-Timothy Radrickson. I-I tossed your food out to make room. I am sorry. I am so sorry. I have no money to pay you back. I just h-h-had to leave. They were after me. I owed money. I had nothing. Please. Any other Republic world will do. I just need to get away. I just need to start anew. I'll p-p-pay you back I swear. I swear. I will. I swear."`
			choice
				`	"Fine. I'll drop you off at the next port."`
					goto port
				`	"Is there somewhere you'd like to go?"`
					goto somewhere
				`	"I know where you're going. The airlock."`

			action
				fail
			`	The feeble man screams as you drag him out of the crate and toward the airlock. He ineptly squirms as you shove him pass the bulkhead seals. He falls to his knees, shudders with terror, and his face leaks tears. You close the airlock, silencing his shrill screams, and hit the button to open the outside airlock. The man is promptly sucked out into cold, cruel space and crystallizes before becoming this system's newest comet.`

			label port
			action
				fail
				set "timothy radrickson: port"
			`	Tears squeeze out the man's sad eyes. "Thank you", he whispers, "thank you."`

> 			`	You get the man cleaned up and settled, and head then go back to running your ship.`

Author: I think adding "the" and "of" just reads very clunky.  But I agree head should be removed.

mission "Timothy Radrickson 2: Interlude"
	landing
	to offer
		has "Timothy Radrickson 1: Stowaway: failed"
		has "timothy radrickson: port"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	on offer
		conversation
			`Blinking at the light, Timothy shuffles toward the exit. "Th-th-thank you. Thank you so much," Timothy mumbles.`
			`	Timothy opens the hatch and looks back at you. "You s-s-saved my life. I won't forget. I won't."`
			`	With that, Timothy exits your ship and begins his new life. You hope everything works out for the best.`

mission "Timothy Radrickson 2: Good Interlude"
	landing
	invisible
	to offer
		has "event: timothy radrickson: on rand"
	on offer
		payment 50000
		event "timothy radrickson: governor" 3600
		conversation
			`After your ship touches down you notice a light blinking on your dashboard. You have a cached message waiting for you to play.`
			choice
				`	(Play the message.)`

> 			`	The screen lights up with the familiar, fuller face of a healthy Timothy Radrickson.`

Author: Did you decide later in the review this change was unnecessary?

			`	Beyond the suddenly green hills sits a huge sprawling complex. You recognize a few massive structures outside the building as artificial gravity drives; the same sort of drives ships use to maintain gravity onboard, but for an entire estate.`
			choice
				`	(Say nothing.)`
					goto nothing
				`	(Ask about the artificial gravity drives.)`

			`	Regina flashes a toothy smile. "The gravity here, it is... taxing. The governor's mansion, and a few other important buildings and housing complexes, use artificial gravity drives to make things more manageable for us hard working 'servants of the state.'" The way she says "servants of the state" makes it sound like some kind of inside joke. She turns away from you before you have a chance to say anything else.`

			label nothing
			`	The vehicle pulls up to a grandiose entrance near the center of the huge structure. An attendant opens the door, and you're ushered outside the vehicle.`
			`	Regina does not get out. She smiles politely through the opened door. "Only you were invited to enter the Governor's Mansion. I am afraid this as far as I am allowed to go. Take care, Captain <last>, it truly was an honor meeting you.`

> 			`	With that the door shuts, and a few moments later the black vehicle departs, leaving you and the attendant at Mansion's entrance.`

Author: I think mansion should be capitalized still since its' the Governor's Mansion', and not just a mansion.

			`	"I am sorry I can't offer you more help." He stands up and hits the button opening up the exit door. "But, please, save that fleet."`
				accept

	on decline
		event "timothy radrickson imprisoned" 400

mission "Timothy Radrickson 3b: Pirate Base"
	landing
	name "Defeat the pirate fleet"
	description "Destroy the pirate fleet that is holding Rand's fleet hostage, then try and recover the fleet."
	source "Greenrock"
	clearance
	to offer
		has "Timothy Radrickson 3a: Rescue the Convoy: done"
	on offer
		conversation
			`	Your ship skims Greenrock's choppy atmosphere and your terrestrial navigation computer suddenly hums to life as it picks up the lost freighter fleet's tracking beacon. It locks on to a large and well guarded underground bay on the very fringes of the spaceport. Close enough to do business, but far enough away to maintain strict security.`

> 			`	As you approach the bay your computer is able to make out numerous parked pirate vessels, and large defensive turrets just outside the bay's underground blast doors. They haven't realized you're hostile yet and you have an opportunity to seize the initative and open fire on them.`

Reviewer 4: ```suggestion
			`	As you approach the bay your computer is able to make out numerous parked pirate vessels, and large defensive turrets just outside the bay's underground blast doors. They haven't realized you're hostile yet and you have an opportunity to seize the initiative and open fire on them.`
```

			choice
				`	(Open fire on the parked pirate vessels.)`
					goto fire
				`	(Hail the pirates and use diplomacy.)`
			action
				set "big greenrock pirate fleet"
			`	You hail the pirates imploring them to consider a diplomatic solution. Their only response is to launch their entire fleet after you.`

> 			`	Your forced back into space to fight them.`

Reviewer 4: ```suggestion
			`	You are forced back into space to fight them.`
```

				launch

			label fire 
			action
				set "diminished greenrock pirate fleet"

> 			`	Your ship lets loose a withering barrage that destroys multiple pirate vessels and disables others. You then pull up back into space as the surviving ships are sent after you.`

Reviewer 4: ```suggestion
			`	Your ship lets loose a withering barrage that destroys multiple pirate vessels and disables others. You then pull back up into space as the surviving ships are sent after you.`
```

			`	In the ensuing chaos you're practically ignored as you approach the secluded bay.`
			choice
				`	(Enter the bay.)`

			`	Inside the bay's walls you can make out the moored and apparently intact freighter fleet. The ships are surrounded by various loaders, but they appear to have been abandoned before they could begin the process of offloading the goods. The pirates must have all raced to their ships when you entered the system.`
			`	"And where do you think you're going?" a voice barks from the control room. Not a moment later, three armed guards appear with their weapons trained on you.`
			choice
				`	"My mistake. I must have parked my ship somewhere else."`
				`	"What a lovely little bay you have here. I was just telling my partner we should buy one just like this."`
				`	"Didn't you guys hear what's going on? Let's loot this stuff before anyone comes looking for it!"`
				`	"Is this where the bathroom is?"`
				`	(Say nothing.)`

> 			`	Before the pirate guards have a chance to respond there's three quick thud sounds in succession and they fall lifelessly to the floor. From behind them a woman and couple of men in red jackets appear.`

Reviewer 4: ```suggestion
			`	Before the pirate guards have a chance to respond there's three thuds in quick succession and they fall lifelessly to the floor. From behind them a woman and couple of men in red jackets appear.`
```

			`	The woman, who is wearing a captain's hat with two H's next to each other looks at you and then nods. "Thanks for the distraction. Been waitin' for the right moment to move."`
			`	She flashes you a reluctant smile. "Name's Hrodny, Capt'n Hrodny of Hroar's Haulers." She looks back at the men accompanying her, whistles, and points toward the ships. They turn and run toward them."`

> 			`	Hrodny turns her green piercing green eyes back on you. "We can get 'em in the air, but we can't fight past whatever is out there. I'm presuming you're not a lost tourist and are in fact our cavalry?"`

Reviewer 4: ```suggestion
			`	Hrodny turns her piercing green eyes back on you. "We can get 'em in the air, but we can't fight past whatever is out there. I'm presuming you're not a lost tourist and are in fact our cavalry?"`
```

			choice
				`	"Don't worry. I took out the fleet, and I'll escort you to Wayfarer."`
				`	"It's your lucky day, I'm Captain <last>, of the <ship>. Getting you to Wayfarer will be a piece of cake."`
				`	"I've been sent to get you to Wayfarer, and that's what I will do."`
				`	"You do your job. I'll do mine."`

			`	Hrodny gives you a feral grin. "Great. We weren't too keen on stickin' around these parts anyhow."`
			`	She looks back at her ships, where there is now a flurry of activity. "Ya know, those pirates parked us right in the middle of their treasure trove. It'd be rude not take advantage of their hospitality." You notice the loaders behind Hrodny whirring around and moving all sorts of goods into the ships.`
			`	Hrodny winks at you. "Hazard pay.`
			`	"You head back your ship, by the time you get there we'll be ready to launch. We'll follow ya out."`
			`	She gives you a final mock salute and then turns and jogs toward her ships. There's nothing for you to do but head back toward yours.`

	on complete
		conversation

> 			`The freighter fleet sets down to some minor fanfare; there're likely more than a few middlemen here who would have been hard-pressed by the loss of the fleet's valuable cargo.`

Reviewer 4: ```suggestion
			`The freighter fleet sets down to some minor fanfare; there are likely more than a few middlemen here who would have been hard-pressed by the loss of the fleet's valuable cargo.`
```

			`	"By way of thanks," he continues, "I've included 50,000 credits with this message. That should cover the cost of transporting to me to Rand, but it doesn't come close to what I really owe. Which is, in all honesty, my very life."`
			`	He pauses, as if he's trying to remember something. "Anyways, thank you again, and know I'll never forget what you did for me."`
			`	Timothy waves and then the video ends.`
				decline

event "timothy radrickson: governor"

mission "Timothy Radrickson 3: Governor Notification"
	name "Visit the Governor on <planet>"
	description "The governor of <planet> has invited you to speak with them."
	to offer
		has "event: timothy radrickson: governor"
	destination "Rand"
	landing
	on offer
		conversation
			`An indicator light is blinking on your main panel. There is a cached message waiting for you to play.`
			choice
				`	(Play the message.)`
			`	A young woman in very professional clothing appears on the screen. "Captain <last>, I am pleased to inform you that the governor of Rand requests your presence. Please come to Rand at your earliest convenience.`

	on complete
		conversation

> 			`Upon landing you are met by a squat, heavy-set woman in her late 40s. She reaches out a calloused hand to shake yours. "Welcome to Rand, Captain <last>. It is a pleasure, and honor, to have someone of your status here on our little world. I am Regina, the Mayor of the spaceport town, and one of the major company-bosses here. I've been instructed to escort you to the Governor's mansion. I have a vehicle waiting for you just outside the spaceport. Meet me there when you're ready."`

Author: I guess. But this is 10 years into the game. Who plays that long with out accomplishing anything?

> 			`Upon landing you are met by a squat, heavy-set woman in her late 40s. She reaches out a calloused hand to shake yours. "Welcome to Rand, Captain <last>. It is a pleasure, and honor, to have someone of your status here on our little world. I am Regina, the Mayor of the spaceport town, and one of the major company-bosses here. I've been instructed to escort you to the Governor's mansion. I have a vehicle waiting for you just outside the spaceport. Meet me there when you're ready."`

Author: Okay. After talking this over with you I've included a requirement for the player to have completed the main quest before the 2nd phase of this quest launches.

			`	Wynnafrith says a few quiet words into an intercom, and a door behind her opens. "The Governor is ready for you."`
			choice
				`	(Enter the room.)`

			`	You enter a large richly furnished room. At the center of the room is an imposing desk, with a slightly graying man seated behind it. There is something immediately familiar about his face.`
			`	"Ah, Captain <last>. At last we can meet face to face, and I can thank you properly." The Governor stands. "You may not remember me, it has been some years since you last laid eyes on me, and," the governor gestures to himself, "I looked quite different back then."`
			`	"I was a stowaway on your ship. You found me in a food crate, and you took me here. I'm Timothy Radrickson, Governor Timothy Radrickson now."`
			`	Timothy sits back down and indicates that you should do the same. "It took me some time. But through some good decisions, and quite a bit of luck, I've been able to claw my way to a level of respectability and wealth. I've really been able to make something of myself."`

> 			`	His face opens up into a large grin. "Then again, compared to what you've done in all this time, my successes hardly register. Still, as meager as they may be, I owe it all to you and your kind gesture. Which is why I've invited you here, to thank you and pay you back properly."`

Author: This is 10 years after the first mission in the campaign. I think it's pretty reasonable to assume at this point.

> 			`	His face opens up into a large grin. "Then again, compared to what you've done in all this time, my successes hardly register. Still, as meager as they may be, I owe it all to you and your kind gesture. Which is why I've invited you here, to thank you and pay you back properly."`

Author: Added a requirement for the player to complete main plot before this mission is offered.

			choice
				`	(Do a surprise attack!)`
					goto surprise
				`	(Get closer.)`
				`	(Hail the pirates and use diplomacy.)`
					goto diplomacy
				`	(This is too dangerous. Leave.)`
					flee

			`	Getting as far as you can to the base without compromising your position, you notice the base is surrounded by an array of laser turrets and ground-to-air missile systems, which makes further approach highly risky. Furthermore, the ships outside the base appear to be highly modified. You don't see any other entrance to the base, however; it looks like you'll have to force the pirates out, one way or another.`
			`	It still seems as if the pirates don't see you. With a little movement, you could strafe the encampment and take out a fair few of the parked ships if you can avoid the turrets.`
			choice
				`	(Strafe the parked pirate vessels.)`
				`	(Hail the pirates and use diplomacy.)`
					goto diplomacy
				`	(This is too dangerous. Leave.)`
					flee

			action
				set "diminished greenrock pirate fleet"

> 			`	Carefully maneuvering into position, your ship flies out of the hills that hid it and lets loose a withering barrage over the parked pirate vessels. You manage to destroy a number of the ships while their shields are down, before the defense turrets force you to fly back up into space as the surviving ships sortie after you.`

Author: I don't know how to do this.

			`	She gives you a mischievous grin. "Apologies, Captain <last>. I forgot to mention the Governor was holding a little get together. Please follow me."`
			choice
				`	(Follow Wynnafrith.)`
			`	She leads you through an army of valets, waiters, and attendants as you weave in and out of the various rooms and halls. Many of the guests appear to recognize you and offer you earnest congratulations. Eventually you are ushered to the table of honor where Wynnafrith introduces you to a half dozen company-bosses, senior executives, political operatives, and the like. You notice Regina here as well, but she is seated at a table in the back, on the fringes of the upper crust of society.`
			`	Not long after being seated Governor Timothy appears and after making small talk with many of the gathered illuminaries sits beside you, "I have something to admit." He says in quietly to you.`
			`	He gives you a sheepish smile, "This is a celebration in your honor, Hero of Rand."`
			choice
				`	"Hero of Rand?"`
				`	"What do mean?"`

> 				`	"So you lied to me."`

Reviewer 5: Maybe provide an option for the player to skip the party if they don't feel like schmoozing with the partygoers?

Author: Given how you were driven to and deposited at the mansion, I think you're stuck there until the event concludes.

Author: Alright, I have added an option to leave the party and sit by yourself in room instead.

			`	He shrugs, "I just thought it would be more fun this way. Who doesn't like surprises?"`
			`	"Without you the entire planet would have defaulted. Our creditors were circling like vultures eager to assume ownership of all our production capabilities, and turn population into endentured servents. The whole world would have been their debtor's prison."`
			`	He pauses to consider his words,"I mean, the language they would have used would be different, but that is what it would have amounted to. But you saved us, and more than that, we're now quite a bit more wealthy than we were before."`
			`	Timothy snatches a glass and holds it up to you, "From ruin to riches! The Hero of Rand.!"`
			`	A few others lift their glasses as well. They clink and Timothy then gulps the whole thing down, after that the group breaks a part into smaller conversations.`

> 			`	Eventually, the party settles down and Timothy leaves you to step up to the podium. He launches into a speech highlighting your career. The great changes you have wrought, the battles you have won, the legends of your ship. The crowd eats it all up, many having heard of some, or all of these exploits. But then Timothy takes a more somber note. He talks quietly about the troubles on his old world. On how as a starving, homeless fugitive he had recklessly hidden aboard the first ship he could find. How you had found him, half dead, in an empty crate. And how, instead of tossing him out an airlock, or dropping off at the nearest world, you took him where he asked. How you took care of him, and how through your generosity, you had given him a second chance.`

Author: It is not. As before though, this happens 10 years after the initial mission, so for the player to be basic at this point is very unlikely.

					goto news
				`	(Ignore it.)`
			`	You keep walking. It's probably nothing, and if it isn't, well it isn't worth your time.`
				decline
			label news
			`	You turn to check out the screen. The talking heads are discussing some sort of huge scandal and the arrest of a prominent governor.`
			`	"That's right, Ann," the anchor on the screen says. "With 8 counts of tax evasion, racketeering, and various corruption charges, Timothy Radrickson - former Governor of Rand - will likely spend the next 20 years behind bars."`
			`	"Where he's going there will be no bars, Tom," Ann quips. "They plan to send him to a work detail on Oblivion."`
			`	Tom gives that sort of news anchor laugh that's neither committal nor haughty. "In that case, forget what I said about 20 years, anything more than 5 years on Oblivion is a life sentence."`
			`	Both news anchors laugh, and then segue into a story about highly intelligent ferrets that have learned to sing in German.`
				accept

mission "Timothy Radrickson 5: Oblivion Meeting"
	landing
	to offer
		random < 25
		has "Timothy Radrickson 4: Imprisoned: active"
	source "Oblivion"
	deadline 1
	invisible
	on offer
		conversation

> 			`Your ship descends through the beautiful, toxic horizon and lands in the dirty, dismal spaceport outside of the provincial capital, Tukayyid. The clumped together warehouses they liberally call a spaceport are used as a central clearing house by all on-world industries, and workers from every industry filter in and out of this never-ending boom town. In the shadows, and on the fringes, the sick, and infirm linger, looking for a handout; the price of progress on such a hostile world.`

Reviewer 5: ```suggestion
			`Your ship descends through the beautiful, toxic horizon and lands in the dirty, dismal spaceport outside of the provincial capital, Tukayyid. The clumped together warehouses they liberally call a spaceport are used as a central clearing house by all on-world industries, and workers from every industry filter in and out of this never-ending boom town. In the shadows, and on the fringes, the sick and infirm linger, looking for a handout; the price of progress on such a hostile world.`
```

			`	His voice gets more animated as he speaks and he nearly launches into another coughing fit before catching himself and taking several uneven breaths. "They're watching me. I can't leave with you. But with the right supplies. Right tools. I can make it."`
			`	His red eyes rimmed with bloody tears look directly into yours. "Please. Don't let me die here. At least.. give me a chance."`
			choice
				`	"Alright. What do you need?"`
					goto agree
				`	(Refuse to help him.)`

			`	You shake your head slowly. "I'm sorry, Timothy. But you broke the law, and this is your punishment. My hands are tied. Maybe when you're out we'll reconnect."`
			`	What little strength he has seems to deflate, a crimson tear winds down his cracked face. "When I'm out?" He almost laughs, "When I'm out!"`
			`	He turns his face towards the truck, before looking back to look at you with a face of forfeited hope. "Goodbye, old friend."`
			`	The weak, broken man returns to loading up the heavy equipment. Slower than before.`
				decline

			label agree

> 			`	Looking sideways first, Timothy takes a piece of yellowed paper from his pocket and hastily writes something down on it before handing it to you. "Have to go off world. Here's a list of what I need. Go to Rand first - address on paper. Norwen mines. Old friend. She will point you in right direction."`

Reviewer 5: ```suggestion
			`	Looking sideways first, Timothy takes a piece of yellowed paper from his pocket and hastily writes something down on it before handing it to you. "Have to go off world. Here's a list of what I need. Go to Rand first - address on paper. Norwen mines. Old friend. She will point you in the right direction."`
```

			`	She gives you a nonchalant shrug. "If that's all you have to say, then I am afraid there is no good reason to let you leave here alive." She favors you with a brief smile. "Operational security, you know how it goes."`
			`	She switches off the safety.`
			choice
				`	"Wait! Timothy sent me!"`
					goto sayso
				`	"Take your best shot."`
			`	She does indeed take her best shot, and the sudden hole in your torso proves to be quite terminal.`
				die

			label sayso
				`	She shakes her head. "Well why didn't you just say so! What does our Governor ask of us?" She holsters her sidearm.`

			label Timmy
			`	Union-Colonel Wynnfrith's eyes get misty as you tell her about Timothy's plight. She shakes her head a few times and then takes the list from you and carefully scrutinizes it.`
			`	"Ya know, to us he's still the Governor. Don't believe what the company-bosses and their paid politicians tell you. He was a good man." She wipes her eyes for a moment. "All he was trying to do was improve the conditions for workers here on Rand. He'd come up from the mines and he knew what it was like."`

> 			`	Her free hand tightens into a ball. "The big men didn't like that. But what about the profit margins? They tried to bribe him, to offer him anything he wanted, to just leave things well enough alone. To let the exploitation continue." She says the last words through clenched teeth.`

Reviewer 5: This implies a level of corruption on Republic worlds that hasn't been seen or implied before in the game. You're telling me that the mining companies on Rand are so powerful that they're able to make up all these fake corruption charges out of thin air and pin them on someone as high up in the government as a planetary governor? Frankly I'd be skeptical of this playing out on a Syndicate world, let alone a Republic one.

Author: Rand is a world that exists for the sake of resource exploitation. That means that people with the most power there would be the companies doing the exploitation. Not only that, but the majority of its populace is transitory. If a governor or civic leader fails to play friendly with the people with all the power. I believe them turning on him is quite realistic. I've worked in the oil fields of North Dakota, I've watched as native lands failed to stop pipelines being implemented through their reservations, I've personally cleaned up oil spills and recorded deaths that never got reported on to anyone. If you think the leading politicians in North Dakota weren't on the same side as the oil companies, I got a slew of bridges to sell you. 
Given the FW issues with the Republic, and the republics relationship with the Syndicate, I think its more than fair to assume that this sort of behavior is not something humanity has grown out of by the year 3000.

Finally, maybe Wynnfrith's side of the story isn't the whole story. Maybe all the charges aren't out of thing air?

Reviewer 5: This is analogous to the oil companies of North Dakota somehow singlehandedly managing to depose the state's governor. I don't deny there's corruption, but on this scale?
If Radrickson tried to improve working conditions on Rand, wouldn't the mining companies just move to any number of the other poor mining planets in human space and set up shop there, instead of hatching this elaborate plan to depose the planet's governor while somehow keeping it under wraps from the rest of the Republic?

Author: I don't think so. Imagine all the investments they've made in developing the infrastructure for resource extraction. To just stop all that would be a huge expense and loss of investment. How much easier to just get rid of the man causing trouble? I would imagine though, as we talk through this, for Tim to get where he is he would had to do some questionable things, and so it would make sense for the companies to have dirt on him. Build a bigger lie around the truth, and have him deposed and replaced with someone more malleable.

Reviewer 3: By the way, it's probable that Rand is part of the FW by the time the player gets this mission. (We could also additionally make Timothy's governorship occur after FW and make it clear.)

Reviewer 5: > I think it could be altered somewhat to include hints that Tim was not perfect, or another perspective on that. In order to become governor he had to first get the support of the companies, and to do that he must have compromised himself. 

Yeah, I think this is more plausible. The charges are real or at least have some basis in reality, and the mining companies conveniently dredge up evidence for those charges when Radrickson stops playing nice with them.

Reviewer 6: Not to disagree with the suggested alteration, which is fine, but:
> This implies a level of corruption on Republic worlds that hasn't been seen before in the game.

The Republic already has 4 distinct intelligence organisations inside it: Deep Security, Republic Intel, Navy Intel, and Syndicate Internal Affairs (which is only internal in the same way that the Syndicate is internal to the Republic).

This doesn't happen in a low-corruption environment.

The way things are presented with the Paradise Worlds and the Syndicate, it's very clear that individual worlds have quite a lot of internal legal autonomy - which means they have quite a lot of scope for corporations to have direct influence over their political processes.
Case in point, the Syndicate is a corporation that literally is the political entity for a dozen worlds.

It's not a stretch to believe this is occurring. This is just a little more direct about it than other things are.

That said, I do like the suggested alteration - it's a little more grey, and a little more in line with characters being unreliable narrators.

Reviewer 5: Hmm, to be honest I saw the Republic proper (outside of the Syndicate or Deep) as being more analogous to the modern-day US, with a structure that gives the states/planets autonomy while still having ultimate authority and oversight derive from the federal government. With Grey's interpretation (which upon reflection is probably more accurate to the actual lore of the game) the Republic's governmental structure is more similar to the modern EU.

Author: I have added in a line when you sit by yourself at the party. Another line when Tim addresses you in his office, and finally altered Wynnfrith's line to acknowledge that Tim's hands weren't clean. 

Let me know what you think and if this is enough.

			choice
				`	(Open fire on the Pirate Bay Defenses.)`
					goto fire
				`	(Attempt to use diplomacy.)`

> 			`	You hail the pirates imploring them to consider a diplomatic solution. There only response is to launch their entire fleet after you.`

Reviewer 2: ```suggestion
			`	You hail the pirates imploring them to consider a diplomatic solution. Their only response is to launch their entire fleet after you.`
```

			`	Your forced back into space to fight them.`

			label fire #Bene - I don't know how to connect the two different options to two different fleet spawns 

> 			`	Your ship lets loose a withering barrage that destroys or disables multiple pirate vessels. You then pull up back into space as the surviving ships are sent after you.`

Reviewer 2: ```suggestion
			`	Your ship lets loose a withering barrage that destroys multiple pirate vessels and disables others. You then pull up back into space as the surviving ships are sent after you.`
```

mission "Timothy Radrickson 1: Stowaway"
	invisible
	landing
	source
		near "Zeta Aquilae" 3 10
		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
	destination "Rand"
	to offer
		random < 10
	on enter

	# You will see this conversation after the first jump you make.
	on enter
		conversation
			`Upon completing your jump, one of your internal sensors begins to beep. You flick off the siren and check the diagnostic report: apparently there is a non-negligible heat source detected in the food storage of your ship.`
			`	As problems go, it is not a particularly alarming one: there are plenty of reasons there may be a heat spike in storage, and most of them would not require the captain of the ship to investigate.`
			`	It is unusual, however.`
			choice
				`	(Investigate.)`
					goto check
				`	(Ignore the sensor.)`

			action
				fail
				event "timothy radrickson: mummy" 41
			`	It's almost certainly nothing, and you have more important things to do than chase down every little sensor glitch on your ship.`

			label check

> 			`	After a short search, and the rearranging of quite a few deceptively heavy boxes, you are able to track the heat signature to a foodstuff crate you recently took on. It has yet to be opened, but you can already tell its seal has been tampered with.`

Reviewer 7: ```suggestion
			`	After a short search, and the rearranging of quite a few deceptively heavy boxes, you are able to track the heat signature to a foodstuff crate you recently took on. It has yet to be opened, but you can tell its seal has already been tampered with.`
```

			choice
				`	(Open the crate.)`
					goto crate
				`	(Dump the crate out the airlock.)`

			action
				fail

> 			`	As the crate tumbles out into lifeless space your onboard sensors quickly note the anomalous heat signature disappear. Whatever was in that crate is no longer your problem, and probably won't be anyone else's either.`

Reviewer 7: There’s no ending here, so will that mean it goes to `label crate`?

Author: i added a decline to after that. Though I am not sure what bene was doing with the action fail stuff so hopefully I didn't mess that up.

Reviewer 7: I think the action fail part only happens after a day passes. `decline` the way you put it in should work immediately.

			label crate

> 			`	You pry open the crate. It should be full of meals for you and your crew, but after removing the lid you discover a dirty disheveled man.`

Reviewer 7: ```suggestion
			`	You pry open the crate. It should be full of meals for you and your crew, but after removing the lid you discover a dirty, disheveled man.`
```

	on complete
		conversation
			`Upon landing you are met by a squat, heavy-set woman in her late 40s. She reaches out a calloused hand to shake yours. "Welcome to Rand, Captain <last>. It is a pleasure, and honor, to have someone of your status here on our little world. I am Regina, the Mayor of the spaceport town, and one of the major company-bosses here. I've been instructed to escort you to the Governor's mansion. I have a vehicle waiting for you just outside the spaceport. Meet me there when you're ready."`

mission "Timothy Radrickson 3a: Rescue the Convoy"
	name "Rescue <origin> convoy"
	description "<origin>'s largest ever convoy was intercepted on its way to Tarazed. Merchants report that a large pirate fleet escorting a freighter convoy passed by Lesath, and it looks like they went to <destination>. Governor Tim has asked you to try and save it before the pirates can plunder it completely."
	source "Rand"
	destination "Greenrock"
	deadline 7
	clearance
	to offer
		has "Timothy Radrickson 3: Invitation to Rand: done"
	on offer
		conversation

> 			`Regina, her driver, and a black sleek vehicle are all waiting for you outside the worn down and substandard spaceport. You slide into the comfortable seating and look out the window as the dry, ugly landscape of Rand passes you by. From time to time you see a strip mine or a camp of workers' barracks, but rarely anything else. Behind your vehicle there is a huge plume of half-frozen dust. Apparently low gravity and dust is a terrible combination.`

Author: So - I agree it would be cool. My understanding is to accomplish this I would need to create branches for every time Rand's environment is discussed and have two different descriptions for each. 

I am not really good at branches and I'd need to see if I could get Bene back to help me with that. And.. it may be a decent amount of work. 

So my question is this: Is Rand terraformed enough by the missions that it deserves two very different descriptions or is it a more gradual/incomplete change where at the time the differences wouldn't be pronounced enough yet for the player to clearly experience them?

				`	(Ask about the artificial gravity drives.)`

			`	Regina flashes a toothy smile. "The gravity here, it is... taxing. The governor's mansion, and a few other important buildings and housing complexes, use artificial gravity drives to make things more manageable for us hard working 'servants of the state.'" The way she says "servants of the state" makes it sound like some kind of inside joke. She turns away from you before you have a chance to say anything else.`

			label nothing
			`	The vehicle pulls up to a grandiose entrance near the center of the huge structure. An attendant opens the door, and you're ushered outside the vehicle.`
			`	Regina does not get out. She smiles politely through the opened door. "Only you were invited to enter the Governor's Mansion. I am afraid this as far as I am allowed to go. Take care, Captain <last>, it truly was an honor meeting you."`
			`	With that the door shuts, and a few moments later the black vehicle departs, leaving you and the attendant at the mansion's entrance.`
			`	The attendant bows deeply. "Please, Captain <last>. If you would follow me." He turns and heads into the building.`
			choice
				`	(Follow the attendant.)`

> 			`	Upon entering the mansion you feel your weight shift. It's still a little less than 1 g, but close. There's a young well-dressed woman, the same who you saw in your message, waiting for you inside. The attendant leaves you with her.`

Author: I think given the genre, it's appropriate.

> 			`	Upon entering the mansion you feel your weight shift. It's still a little less than 1 g, but close. There's a young well-dressed woman, the same who you saw in your message, waiting for you inside. The attendant leaves you with her.`

Author: I don't think it's supposed to be capitalized.

			`	The woman smiles pleasantly. "We are all so pleased you could make it here. Forgive me for not meeting you outside, but I find the gravity of this world distressing. I am Wynnafrith, the governor's executive assistant, and will be pleased to announce that you are here.`
			`	Wynnafrith says a few quiet words into an intercom, and a door behind her opens. "The Governor is ready for you."`
			choice
				`	(Enter the room.)`

			`	You enter a large richly furnished room. At the center of the room is an imposing desk, with a slightly graying man seated behind it. There is something immediately familiar about his face.`
			`	"Ah, Captain <last>. At last we can meet face to face, and I can thank you properly." The Governor stands. "You may not remember me, it has been some years since you last laid eyes on me, and," the governor gestures to himself, "I looked quite different back then."`

> 			`	"I was a stowaway on your ship. You found me in a food crate, and you took me here. I'm Timothy Radrickson, Governor Timothy Radrickson now."`

Author: He's not stating an 'or' here. It's an upgraded statement.

			choice
				`	(Accept the money.)`
					goto moneyaccept
				`	(Refuse the money.)`

			`	You begin to refuse the generous offer, but Timothy holds up a hand to stop you. "I am doing quite well, and this discharges a debt that has been weighing on me. Besides, the money's already been sent!"`

			label moneyaccept
			`	Timothy pulls out a bottle beneath his desk and two glass cups. "To celebrate our reunion."`
			`	He pours the brown liquid in the cups and pushes one to you. "Things have gone well for me, and are only going to get better from here on out."`
			`	He takes a sip and says, "We've just launched our largest freighter fleet to date." He sits back in his desk, relaxed, and gets a dreamy look in his eyes. "When it returns from Wayfarer, the government of Rand will finally have enough credits to make some real changes around here. Give something back to the working man."`
			choice
				`	(Congratulate Timothy.)`
				`	(Take a sip from your glass.)`
				`	(Complain about the living standards of workers on Rand.)`
				`	(Complain about hand-outs for the poor of Rand.)`

> 			`	His eyes refocus on you. "You'd be proud, <first>. I am going to make a real difference here..."`

Author: I just thought his response would be the same regardless of what you said. 

If say nothing, or give him a congrats, he says you'd be proud because he's done good work.

If you complain about something, he's also going to say you'd be proud because he's making changes.

	on offer
		conversation
			`Your ship skims Greenrock's choppy atmosphere and your terrestrial navigation computer suddenly hums to life as it picks up the lost freighter fleet's tracking beacon. It locks on to a large encampment situated in a valley a bit under fifty kilometers northeast of the planetary spaceport - close enough to do business, but far enough away to maintain strict security.`
			`	As you approach the beacon's ping from the safety of the hills around the valley, you survey the base from afar. It clearly appears to be built around a set of underground blast doors, from where the beacon - and presumably the fleet - is located. Your ship also identifies numerous parked pirate vessels inside and outside the base.`
			`	The pirates haven't realized you're hostile yet, and you have an opportunity to seize the initiative and do a surprise attack. Or, you could get closer and continue surveying the base to see if there's any way in.`
			choice
				`	(Do a surprise attack!)`
					goto surprise
				`	(Get closer.)`
				`	(Hail the pirates and use diplomacy.)`
					goto diplomacy
				`	(This is too dangerous. Leave.)`
					flee

> 			`	Getting as far as you can to the base without compromising your position, you notice the base is surrounded by an array of laser turrets and ground-to-air missile systems, which makes further approach highly risky. Furthermore, the ships outside the base appear to be highly modified. You don't see any other entrance to the base, however; it looks like you'll have to force the pirates out, one way or another.`

Author: 'towards' is against the rules for ES. I'll replace with toward though.

				launch

	on visit
		dialog `There are still marauder ships flying overhead. Take off and defeat them before finding the fleet.`

	npc kill
		to spawn
			has "big greenrock pirate fleet"
		government "Pirate"
		personality launching nemesis target plunders
		fleet
			names "pirate"
			variant
				"Marauder Falcon (Engines)"
				"Falcon (Javelin)"
				"Marauder Firebird (Weapons)"
				"Firebird"

				"Marauder Raven"
				"Modified Argosy (Gatling)" 2
				"Osprey (Missile)"

				"Hawk (Plasma)"
				"Quicksilver" 2
				"Headhunter"
				"Fury (Gatling)" 3
	
	npc kill
		to spawn
			has "diminished greenrock pirate fleet"
		government "Pirate"
		personality launching nemesis target plunders
		fleet
			names "pirate"
			variant
				"Marauder Falcon (Engines)"
				"Falcon (Javelin)"
				"Marauder Firebird (Weapons)"
				"Firebird"

				"Modified Argosy (Gatling)"
				"Osprey (Missile)"

				"Hawk (Plasma)"
				"Fury (Gatling)"

mission "Timothy Radrickson 3c: Escort the Convoy"
	landing
	source "Greenrock"
	destination "Wayfarer"
	to offer
		has "Timothy Radrickson 3b: Pirate Base: done"
	on offer
		conversation

> 			`The sky burns red as detritus from the fierce battle rains down on Greenrock. The destruction of a major pirate gang's fleet has resulted in panic and chaos over the region as rival pirate groups battle it out in the skies to claim the defeated's turf. Although a few ships take pot shots at you as you descend into the atmosphere, everybody is too busy brawling to notice you as you approach the pirates' encampment from where you still register the ping of the beacon.`

Author: Who doesn't?

			`	As you exit your ship a woman wearing a captain's hat approaches you and then flashes a reluctant smile at you. "Name's Hrodny, Capt'n Hrodny of Hroar's Haulers. You probably heard Alan on the radio," she says, flicking her eyes to the control room.`
			`	"Well, thanks for the distraction. Once you got the fleet out of here we overpowered what little detail they kept here and took control of the turrets." She looks around to the blast doors where the depowered turrets once guarded. "Been waitin' for the right moment to move outta this place."`
			`	Hrodny turns her piercing green eyes back on you. "We got 'em in here easy, but we can't fight past whatever is out there. I'm presuming you're not just a lost mercenary and are in fact our cavalry?"`
			choice
				`	"Don't worry. I can escort your fleet to Wayfarer."`
				`	"It's your lucky day. Getting you to Wayfarer will be a piece of cake."`
				`	"I've been sent to get you to Wayfarer, and that's what I will do."`
				`	"You do your job. I'll do mine."`

			`	Hrodny grins. "Great. We weren't too keen on stickin' around these parts anyhow."`

> 			`	She looks back at her ships, whistles, and points toward the ships. They turn and run toward them, preparing them for takeoff. She then looks back at you. "Ya know, those pirates parked us right in the middle of their treasure trove. It'd be rude not take advantage of their hospitality." You notice the loaders behind Hrodny whirring around and moving all sorts of goods into the ships, while a number of crew are draining the base's deuterium tanks and filling up the ships inside.`

Author: Changed it to a, "a group of haulers"

				goto nextup

			label shake
			`	Her hand vigorously shakes yours. "Gotta say you handled yourself well out there. Always good to see a Capt'n worth more than the dust on their boots." She gives you an approving nod.`

			label nextup
			`	"I got word from Rand they are more than a little pleased with your heroics. They'd like you to head on over there when you can. Probably have some credits or a photo op or something fancy and tedious waiting for you."`
			`	She shrugs, and winks, "Anyways, I got plenty of work that needs doing. Take care of yourself, Capt'n."`
			`	With that she heads back to her freighter fleet.`

	npc accompany save
		government "Merchant" # 3800 cargo capacity
		personality escort timid
		ship "Behemoth" "H.H.H. Gullveig's Revenge"
		ship "Hauler" "H.H.H. Hard and Fast"
		ship "Hauler" "H.H.H. Red Rambler"
		ship "Hauler III" "H.H.H. Heavy Haulage"
		ship "Hauler III" "H.H.H. Hroar's Heavy Hitter"
		ship "Hauler II" "H.H.H. First Last Chance"
		ship "Hauler II" "H.H.H. Lucky Hauler"

> 		ship "Mule" "H.H.H. Herald of Vermilion"

Author: I don't understand this comment.

> 		ship "Mule" "H.H.H. Herald of Vermilion"

Author: so you're saying the total cargo capacity is 80 tons more?

> 		ship "Mule" "H.H.H. Herald of Vermilion"

Author: and it'll know that belongs to the Mule?

> 		ship "Mule" "H.H.H. Herald of Vermilion"

Author: Okay, I just added the boxwing.

> 		ship "Mule" "H.H.H. Herald of Vermilion"

Author: Got it!

		ship "Bulk Freighter" "H.H.H. Cardinal Conveyor"
		ship "Bulk Freighter" "H.H.H. Muspel Zipper"
		ship "Freighter" "H.H.H. Ruby Rails"
		ship "Freighter" "H.H.H. Dusty Boots"

mission "Timothy Radrickson 3d: Return to Rand"
	landing
	source "Wayfarer"
	destination "Rand"
	to offer
		has "Timothy Radrickson 3c: Escort the Convoy: done"
	on complete
		conversation
			`Instead of Regina you are met by Timothy's executive secretary, Wynnafrith. She is clearly a little uncomfortable in Rand's regular gravity, but is doing her best to push through it.`
			`	"Captain <last>! You're the talk of the world! The savior of Rand." She gives you an open and earnest smile.`
			`	"I've been sent by the Governor to drive you straight to his estate. Please come with me."`
			choice

> 				`	(Go to the Governor's Mansion.)`

Author: If you look up Govneror's Mansions, you'll see they are all capitalized.

> 				`	(Go to the Governor's Mansion.)`

Author: Yeah, a few people have had a crack at this... and I think there has been some back and forth on this. 
Some reviewers have considered that mention of 'The mansion' as needing to be lower case. I think they decided it was referring to the building in general, and not specifically as the Governor's Mansion? 

Honestly, at this point I am confused and afraid to touch it.

			`	Light reflects off the tears that cling to Timothy's eyes as he tells the assembled crowd that, for him, that was your greatest moment, your most heroic action, that nothing you've done, or ever will do, compares to that simple act of mercy. And that this, to him, alone, is what makes you great.`
			`	From here his story shifts to your selfless offer to help save Rand, even though they could promise no reward, of your stunning victory over Greenrock, and your daring liberation of the Fleet on the ground. He ends with you triumphant, and Rand saved.`

> 			`	The assembled guests stand, clap and cheer. They look at you, at your seat of honor, and hold up glasses in your honor. Eventually as the cheers and hoots die down, the Governor declares you a Hero of Rand, and thanks you for your service.`

Author: Thanks, I changed the verbiage.

			`	You nod slowly. You have to admit, he hasn't really been creepy, and it's not like he's been writing you letters every week or anything. It's been almost a decade since you heard from him last.`
			choice
				`	"Thanks for inviting me. I've had a great time."`
				`	(Go back to your ship.)`
					goto ship

			label thanks
			`	"Oh, it's nothing compared to what you did for me. What you saved me from, and what you did to bring me here. And then how you bailed me and Rand out again." Timothy glances away from you, as if embarrassed all he did tonight wasn't enough.`
			choice
				`	(Keep talking with Timothy.)`
				`	(Excuse yourself and go back to your ship.)`
					goto ship

			`	You talk with him for hours. About his life on Rand, the hardships of the mines, how he clawed his way out of it. But also about you, the challenges of running a ship, the battles, the close calls, and the joys of triumph.`
			`	As you talk you come to realize there is a sadness at the edges of Timothy's stories about being governor. His words seem tinged with regret, though he says nothing specifically regretful. You get the feeling he isn't telling you the whole story.`

			label end

> 			`	As the sun rises your conversation winds to a close. Timothy escorts you back to a waiting vehicle and you say your farewells. The dark vehicle drives off into the bright desert, leaving behind an island of green and warmth.`

Author: >>
<<
It's the dusty part of Rand.

In all seriousness, roadrunner mentioned this above... I'll address the larger terraformed question in their comment up there.

				decline

			label ship
			`	Timothy acquiesces and escorts you back to a waiting vehicle, and you say your farewells. The dark vehicle drives off into the bright desert, leaving behind an island of green and warmth.`
				decline

event "timothy radrickson imprisoned"

mission "Timothy Radrickson 4: Imprisoned"
	to offer
		has "event: timothy radrickson imprisoned"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Oblivion"
	deadline 1800
	invisible
	on offer
		conversation
			`As you stroll through the spaceport, a name blaring on a nearby news screen catches your attention.`
			choice
				`	(Watch the news.)`
					goto news
				`	(Ignore it.)`

> 			`	You keep walking. It's probably nothing, and if it isn't, well it isn't worth your time.`

Author: I like the feel of it being more curt and decisive.

			choice
				`	"Alright. What do you need?"`
					goto agree
				`	(Refuse to help him.)`

			`	You shake your head slowly. "I'm sorry, Timothy. But you broke the law, and this is your punishment. My hands are tied. Maybe when you're out we'll reconnect."`
			`	What little strength he has seems to deflate, a crimson tear winds down his cracked face. "When I'm out?" He almost laughs, "When I'm out!"`
			`	He turns his face towards the truck, before looking back to look at you with a face of forfeited hope. "Goodbye, old friend."`
			`	The weak, broken man returns to loading up the heavy equipment. Slower than before.`
				decline

			label agree
			`	Looking sideways first, Timothy takes a piece of yellowed paper from his pocket and hastily writes something down on it before handing it to you. "Have to go off world. Here's a list of what I need. Go to Rand first - address on paper. Norwen mines. Old friend. She will point you in the right direction."`
			`	He abruptly turns away from you and goes back to loading equipment, but slower now, buying for time.`

> 			`	"Eighteen days," he says wearily. "Then I go south. Lost in the maze of rigs." He pauses before saying more quietly, "and then I die." He looks back at you one last time. "Always owe you." His face cracks in one painful looking grin. "Good luck, my good friend."`

Author: Good catch. Fixed it.

			`	In fact, what Ivaldur has done is create a fake person, and this is that person's real ID.`
			`	When he finishes he says nothing and instead basks in a job well done. Finally, he turns to you for payment. <payment> is transferred into his account, and he hands you Tim's brand new ID card.`
			`	"It was a pleasure," he says in a satisfied voice, and bids you well.`
				goto greenoptions
		
			label suit1
			action
				set "suit1"
			`	You find a large stall boasting all the survival equipment an enterprising pirate could ever need. Inside there's various used gear, some more used than others, and only a few well weathered survival suits.`
			`	The proprietor appears out of nowhere and immediately begins flogging his goods to you. "Ah! Yes. With any of these you can survive the harshest of climates!"`

> 			`	He takes one suit off the rack and begins, "Notice the intricate mesh weave on this state of the art..." the suit comes apart at its seams in his hands. The proprietor frowns, but then quickly tosses it to the side.`

Reviewer 5: ```suggestion
			`	He takes one suit off the rack and begins, "Notice the intricate mesh weave on this state of the art-" the suit comes apart at its seams in his hands. The proprietor frowns, but then quickly tosses it to the side.`
```

			`	"A, uh, defunct specimen. Happens from time to time. Rest assured I never would have sold it to you." He looks at you unconvincingly.`
			`	He pulls another suit down which, to his obvious relief, doesn't simply fall apart in his hands.`
			`	"Now as I was saying, top of the line mesh weaves. Great comfort and breathability. For tough spots..."`
			`	You look carefully at the suit as the proprietor's words drift uselessly out of his mouth. It has patches on top of patches, and its seals look worn and corroded. It looks like a death trap.`
			`	"...and look just how clean it looks inside." The proprietor opens up one the seals, revealing quite a bit of dried blood staining its insides. He quickly closes the seal and wipes some sweat from his brow.`

> 			`	"Perfectly normal to find in a suit. Totally unrelated to its operation. You know, pirates, just bleed a lot in general. Part of the lifestyle and all that."`

Reviewer 5: ```suggestion
			`	"Perfectly normal to find in a suit. Totally unrelated to its operation. You know, pirates... just bleed a lot in general. Part of the lifestyle and all that."`
```

				payment -100000
			`	The proprietor quickly rings you up as if worried you'll change your mind at any moment, and gleefully takes your <payment>.`
			`	"Safe travels, privateer," he says as you leave his stall.`
				goto greenoptions

			label getgear4
			action
				set "suit2"
				"timothy radrickson suit" = 1
				payment -250000
			`	The proprietor quickly rings you up as if worried you'll change your mind at any moment, and gleefully takes your <payment>.`
			`	"Safe travels, privateer," he says as you leave his stall.`
				goto greenoptions

			label med1
			action
				set "med1"
			`	There are many stalls loudly advertising all kinds of illegal drugs, and it takes some work to sift through them until you come across one selling legal drugs as well. The man running it is sallow and pock-marked, but his eyes are keen. He almost hisses as he greets you.`
			`	"Whatever you need, wherever you're going. I have you covered."`
			choice
				`	(Ask about detoxification medicine for Oblivion.)`
					goto askdrugs
				`	(Leave.)`

			`	This is not the place for you. You take your leave and return to the emporium at large.`
				goto greenoptions

			label askdrugs

> 			`	The drug-dealer flashes you with his yellow teeth and unnerving smile. "For every kind of toxic environment under any sun." He jolts towards his little wall of drawers, pulling out one labeled "Oblivion detoxifiers." From the drawer he picks a bottle up, checking its label.`

Reviewer 5: ```suggestion
			`	The drug dealer flashes you with his yellow teeth and unnerving smile. "For every kind of toxic environment under any sun." He jolts towards his little wall of drawers, pulling out one labeled "Oblivion detoxifiers." From the drawer he picks a bottle up, checking its label.`
```

			`	You leave the drug-dealer behind and head back into the emporium.`
				goto greenoptions
			
			label getmed
			action
				set "med2"
				"timothy radrickson	medicine" = 2
				payment -50000
			`	"Trust me, it's still good," the drug-dealer says as he takes your <payment>. He hands you the drugs with a smirk.`
				goto greenoptions

			label end
			action
				clear "med1"
				clear "med2"
				clear "suit1"
				clear "suit2"
				clear "id1"
				clear "id2"
			`	It's never wise to linger in a place like this, especially aimlessly. With your business concluded you leave Greenrock's stalls behind.`
				decline

mission "Timothy Radrickson 5e: Back to Oblivion"
	source Oblivion
	to offer
		has "Timothy Radrickson 5a: Timshank Redemption: done"
	on offer
		"timothy radrickson preparedness" = "timothy radrickson	medicine" + "timothy radrickson suit" + "timothy radrickson id"
		conversation
			`Your ship touches down beneath Oblivion's toxic ammonia-blue sky. Night or day, there is a constant flurry of workers loading, unloading, building, and expanding the ever growing boom-town of Tukayyid. You unload all of Timothy's gear onto a dolly and push it out into the busy street.`

> 			`	Your presence goes unnoticed among the general ruckus and you're able to slip into the off-loading warehouses unseen. You find frail Timothy there, leaning over a pile of hose hookups, breathing heavily. His blood red eyes rise to meet yours, and despite their bloody, milky gloss, a glimmer of hope burns within them.`

Reviewer 5: ```suggestion
			`	Your presence goes unnoticed among the general ruckus and you're able to slip into the off-loading warehouses unseen. You find Timothy there, leaning over a pile of hose hookups, breathing heavily. His blood red eyes rise to meet yours, and despite their bloody, milky gloss, a glimmer of hope burns within them.`
```

			`	"You did it?" His words more strained than the last time you spoke. "You got the gear?"`
			`	He looks down at your dolly expectantly.`
			choice
				`	"It's all here."`
				`	"Of course I did."`
				`	"Yep, but you owe me big."`

			`	He nearly drops to his knees and starts sorting through the gear. He takes heart when you tell him about Wynnafirth's gun and bomb, and how highly she still thinks of him.`
			`	"Good secretary. Better friend." He wheezes as he checks out the rest of your gear.`
			branch unprepared
				"timothy radrickson preparedness" < 6

			branch "decently prepared"
				"timothy radrickson preparedness" >= 6
				"timothy radrickson preparedness" <= 7

			branch "well prepared"
				"timothy radrickson preparedness" > 7

			label unprepared

> 			`	As he inspects the suit, drugs, and ID you can see his demeanor shift. He seems to inflate a little.`

Reviewer 5: Did you mean to say deflate here?

Author: yes

Author: Okay, fixed it. Thank you.

			`	Finally, he straightens back up. "You did the best you could. I'm sure." He eyes the gear wearily. "At least it's a chance."`
				goto end

			label "decently prepared"
			`	As he inspects the suit, drugs, and ID you he nods to himself satisfactorily.`
			`	Finally, he straightens back up. "It'll do," he says as he eyes the gear. "It'll do."`
				goto end

			label "well prepared"
			`	As he inspects the suit, drugs, and ID you can see his demeanor shift. He looks less weak, less sickly, and more confident.`
			`	Finally, he straightens back up. "It's perfect." He eyes the gear with awe and shakes his head. "Don't know how you did it. But with this I can't fail."`

			label end

> 			`	Timothy turns from you and uses the dolly to load the gear into one of the trucks in the warehouse. Once done he turns back to you, his eyes are wide, rimmed with bloody tears. "Always at my lowest." He swallows a cough, and then continues. "It's you who saves me. When I am at my lowest. It's you."`

Reviewer 5: ```suggestion
			`	Timothy turns from you and uses the dolly to load the gear into one of the trucks in the warehouse. Once done he turns back to you, his eyes wide and rimmed with bloody tears. "Always at my lowest." He swallows a cough, and then continues. "It's you who saves me. When I am at my lowest. It's you."`
```

			`	Timothy looks like he wants to hug you, but he just nods. He hits a button on the wall and the garage door opens, exposing the warehouse to the night air.`
			`	"Captain," he says in way of a farewell, and then he climbs aboard the truck and drives off.`

mission "Timothy Radrickson 6: News of Death"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	to offer
		has "Timothy Radrickson 5e: Back to Oblivion: offered"
		random < 5
	on offer
		event "timothy radrickson forever"
		conversation

> 			`A spaceport is a symphony of smells, sounds, and images in constant motion. Somewhere in the din, a news report is playing, and a familiar name leaps out at you. Your attention piqued, you turn to watch.`

Author: Is there a way around this? Or do I just specify a bunch of different worlds it could happen at? Maybe core?

> 			`A spaceport is a symphony of smells, sounds, and images in constant motion. Somewhere in the din, a news report is playing, and a familiar name leaps out at you. Your attention piqued, you turn to watch.`

Author: I'll do that.

			`	"News out of Oblivion today. A minor explosion has the claimed the life of Timothy Radrickson, former disgraced Governor of Rand. Sentenced to prison-labor on Oblivion, a hard but fair justice for a man who..."`
			`	The news broadcast continues to blare, but you aren't able to follow anymore than that.`
				decline

event "timothy radrickson forever"

mission "Timothy Radrickson 7: Monk"
	source "New Tibet"
	to offer
		random < 25
		has "event: timothy radrickson forever"
		"timothy radrickson preparedness" > 7
	on offer
		conversation
			`A group of monks passes you by in the spaceport, and one stops to look at you curiously.`
			`	"Are you Captain <last>?" They ask.`
			choice
				`	"Why?"`
					goto why
				`	"Yes."`
					goto why
				`	"None of your business."`

> 			`	"Of course," the monk says, he drops his head and rejoins his brothers.`

Reviewer 5: ```suggestion
			`	"Of course," the monk says. He drops his head and rejoins his brothers.`
```

				decline

			label why
			`	The monk lowers his head in a sign of respect. "Although we live in the mountains and interact rarely with outsiders, we have heard much of your exploits. You are an agent of change, and worthy of consideration."`
			`	"I thank you for giving us much to ponder on. I also have one small request; there is one within our monastery who is a great admirer of yours. It would mean much if you would be willing to meet him."`
			choice
				`	"I'd be honored to meet him."`
					goto whyyes
				`	"Why not. I got nothing to better to do."`
					goto whyyes
				`	"I don't have time for fans."`

			`	"Of course," the monk says, "you are a busy person." He returns to his brothers.`
				decline

			label whyyes
			`	The monk beams. "Wonderful. It'll be a bit of a hike, but the views are worth it."`

> 			`	He leads you into the mountains beyond the spaceport. You travel up various narrow passes, and steep cliff sides. It's a difficult trail, but the scenery is remarkable. Your views are filled with broken mountains, jagged rocks, green vibrant life, and distant valleys.`

Reviewer 5: ```suggestion
			`	He leads you into the mountains beyond the spaceport. You travel up various narrow passes and steep cliff sides. It's a difficult trail, but the scenery is remarkable. Your views are filled with broken mountains, jagged rocks, green vibrant life, and distant valleys.`
```

			`	After several arduous hours, you clear a peak, and discover a vast stone building waiting for you. "We have arrived," the guide says almost anticlimactically. Perhaps the fact he lives here makes it less grand to him than it ought to be.`
			`	Your guide leaves you outside the entrance, and disappears within for a few minutes before returning with another monk.`
			`	The new monk looks at you with an open, serene expression, that quickly breaks into a wide grin. Although much about his face has changed, whether from wear or surgery, you recognize that grin.`
			choice
				`	"Timothy!"`
				`	"Oh, it's you."`

			`	Timothy Radrickson, former prison-worker on Oblivion, Governor of Rand, and stowaway of your ship, hugs you.`
			`	"Captain <last>. I always hoped I would see you again. Given my tenuous legal situation, I can't leave here, and I can't send any messages." He steps back to look at you.`

> 			`	"The years have been kind to you, Captain; you hardly look any different from when we first met! Perhaps I should have invested in ship life, instead of terrestrial!" He laughs at his own joke, but only modestly.`

Reviewer 5: ```suggestion
			`	"The years have been kind to you, Captain; you hardly look any different from when we first met! Perhaps I should have invested in ship life, instead of staying terrestrial!" He laughs at his own joke, but only modestly.`
```

> 			`	"It was difficult for me to speak, when last we met on Oblivion, but you're my oldest and best friend. At my lowest points you were always there for me. I just wanted to thank you for that."`

Reviewer 5: ```suggestion
			`	"It was difficult for me to speak when last we met on Oblivion, but you're my oldest and best friend. At my lowest points you were always there for me. I just wanted to thank you for that."`
```

					goto offer2
				`	(Tell him you don't need the money, but he does.)`
					goto offer2
				`	(Agree with him and rescind your offer.)`
					goto goodbyes
			
			label offer2
			action
				clear "salary: Hero of Rand"
				clear "offers"
			`	As you speak Timothy shakes his head slowly, tears streaming down his face. Finally, as you finish, he wraps his arms around you in an earnest heartfelt hug.`
			`	He tries to speak, but his voice fails him. Instead he just shakes his teary-eyed head at you and gives you a deep solemn bow.`
			`	Timothy stays to watch as you begin your slow descent down the mountain, and back to civilization.`
			decline

mission "Timothy Radrickson 7: Hospice"
	source "New Tibet"
	to offer
		"timothy radrickson preparedness" >= 6
		"timothy radrickson preparedness" <= 7
		random < 25
		has "event: timothy radrickson forever"
	on offer
		conversation
			`A group of monks passes you by in the spaceport, and one stops to look at you curiously.`
			`	"Are you Captain <last>?" They ask.`
			choice
				`	"Why?"`
					goto why
				`	"Yes."`
					goto why
				`	"None of your business."`

> 			`	"Of course," the monk says, he drops his head and rejoins his brothers.`

Reviewer 5: ```suggestion
			`	"Of course," the monk says. He drops his head and rejoins his brothers.`
```

			`	"We have arrived," your guide says, and as soon as you look into the ward he seems to disappear into the crowds of the sprawling hospital.`
			`	In the ward you find a maimed man with burn scars and slightly labored breathing. His face lights up when he looks at you, and despite his scars you recognize him. It's Timothy Radrickson.`
			choice
				`	"Timothy!"`
				`	"Oh, it's you."`

			`	Timothy Radrickson, former prison-worker on Oblivion, Governor of Rand, and stowaway of your ship, beckons you step closer to him with his good hand.`
			`	"Captain <last>. I always hoped I would see you again. Given my tenuous legal situation, not to mention my health, I can't leave here, and I can't send any messages." His eyes linger on you, as though he is taking a mental picture. Or just enjoying the sight of a good friend returned.`

> 			`	"The years have been kind to you, Captain; you hardly look any different from when we first met! Perhaps I should have invested in ship life, instead of terrestrial!" He laughs at his own joke, but only weakly given his condition. He coughs a bit, and takes a few slow deep breaths before continuing.`

Reviewer 5: ```suggestion
			`	"The years have been kind to you, Captain; you hardly look any different from when we first met! Perhaps I should have invested in ship life, instead of staying terrestrial!" He laughs at his own joke, but only weakly given his condition. He coughs a bit, and takes a few slow deep breaths before continuing.`
```

			`	"It's still a little difficult for me to speak, but I feel much better than when last we met on Oblivion. I just wanted you to know you're my oldest and best friend. At my lowest points you were always there for me. So thank you for that."`
			`	You spend the day in the hospital room chatting with with Timothy, before finally having to leave.`
			`	"Thank you for coming here. I wish I had something to give you, but I own nothing anymore, even though I owe you everything. Take care out there, and I'll do my best to keep out of trouble in here."`
			label goodbyes
			choice
				`	(Offer to transfer the Hero of Rand award and stipend to Timothy.)`
					goto offer
					to display
						not "offers"
				`	(Shake his hand and say goodbye.)`
					goto shake
				`	(Give him a hug and say goodbye.)`

			action
				clear "offers"

> 			`	You lean over and give him partial hug in his sick bed. He wraps his one good arm around you and squeezes. When he finally breaks free you see a tear his eyes are blurry with tears.`

Reviewer 5: ```suggestion
			`	You lean over and give him partial hug in his sick bed. He wraps his one good arm around you and squeezes. When he finally breaks free you see a tear.`
```

> 			`	You leave him bedridden in the hospital, and to your surprise find your guide waiting for you outside the room. "His is a tough case. Perhaps one day he will recover enough to leave this place." The monk pauses, thoughtfully. "Or perhaps not."`

Reviewer 5: ```suggestion
			`	You leave him bedridden in the hospital, and to your surprise you find your guide waiting for you outside the room. "His is a tough case. Perhaps one day he will recover enough to leave this place." The monk pauses, thoughtfully. "Or perhaps not."`
```

			`	He leads you back to the train station, and from there you take brief journey back to New Tibet's spaceport.`
				decline

			label shake
			action
				clear "offers"

> 			`	You lean in to shake his good hand, he swallows and clears his throat, trying to keep himself from crying.`

Reviewer 5: ```suggestion
			`	You lean in to shake his good hand. He swallows and clears his throat, trying to keep himself from crying.`
```

			`	You leave him bedridden in the hospital, and to your surprise find your guide waiting for you outside the room. "His is a tough case. Perhaps one day he will recover enough to leave this place." The monk pauses, thoughtfully. "Or perhaps not."`
			`	He leads you back to the train station, and from there you take brief journey back to New Tibet's spaceport.`
				decline
		
			label offer
			action
				set "offers"
			`	Timothy's eyes go wide. "No, you couldn't. It's yours. You've given me too much already."`
			choice
				`	(Tell him about what Wynnafrith said, and how he is the real Hero of Rand.)`
					goto offer2
				`	(Tell him you don't need the money, but he does.)`
					goto offer2
				`	(Agree with him and rescind your offer.)`
					goto goodbyes
			
			label offer2
			action
				clear "salary: Hero of Rand"
				clear "offers"

> 			`	As you speak Timothy shakes his head slowly, tears streaming down his face. Finally, as you finish, he reaches out to you with his one good arm and pulls you in for an earnest heartfelt hug.`

Reviewer 5: ```suggestion
			`	As you speak Timothy shakes his head slowly, tears streaming down his face. Finally, as you finish, he reaches out to you with his one good arm and pulls you in for a hug.`
```

			`	You walk across the forgotten memorial to find a small, flat unremarkable stone, hemmed in amongst other unremarkable stones, pressed into the dead ground. The text laser-chiseled onto it reads, "Timothy Radrickson. Prisoner - 645631-4C"`
			`	That's all. Nothing about his meteoric rise, about being the Governor of Rand, or even how or when his death occurred here. Just another dead prisoner.`
			choice
				`	(Stay a while and contemplate.)`
				`	(Leave.)`
					goto end

			`	The slow moving evening clouds have cast the entire world in a strange purple twilight. Timothy's grave sits before you, nestled in the poison ground, and you alone in the vast empty graveyard.`
			`	Your thoughts drift to the man buried beneath your feet.`
			label stayawhile
			choice
				`	(Think about Timothy's past.)`
					goto past
					to display
						not "past"
				`	(Think about Timothy's rise.)`
					goto rise
					to display
						not "rise"
				`	(Think about Timothy's demise.)`
					goto demise
					to display
						not "demise"
				`	(Leave the graveyard.)`
					goto end

			label past
			action
				set "past"

> 			`	You never knew what Timothy had been fleeing when he first stowed away on your ship. Who he had been, or what he had done that lead him to such dire straits.`

Reviewer 5: All of these descriptions feel better suited for a logbook entry than as part of a conversation.

Author: I am okay with that. I've no idea how the logbook works though, or really understand it/used it on my play through.

Reviewer 5: You can find basic information on how to create logs in a mission by looking at https://github.com/endless-sky/endless-sky/wiki/CreatingMissions. If you look in human missions.txt you can see examples of logs for people you meet during various missions. I'd adapt the stuff you've written here as logs throughout this mission chain, because it feels weird to go through this entire mission chain without Timothy getting mentioned in the logbook.
For example, I'd use lines 9297-8 as log entries for right after you meet Timothy and after he pays you back. If you need help with this I can try making a PR to your branch, but I can't guarantee I'll be able to get it done soon.

				goto nextup

			label shake
			`	Her hand vigorously shakes yours. "Gotta say you handled yourself well out there. Always good to see a capt'n worth more than the dust on their boots." She gives you an approving nod.`

			label nextup
			`	"I got word from Rand they are more than a little pleased with your heroics. They'd like you to head on over to the spaceport when you can. Probably have some credits or a photo op or something fancy and tedious waiting for you."`
			`	She shrugs, and winks, "Anyways, I got plenty of work that needs doing. Take care of yourself, Capt'n."`
			`	With that she heads back to her freighter fleet.`

	npc accompany save
		government "Merchant" # 3880 cargo capacity
		personality escort timid
		ship "Behemoth" "H.H.H. Gullveig's Revenge"
		ship "Hauler" "H.H.H. Hard and Fast"
		ship "Hauler" "H.H.H. Red Rambler"
		ship "Hauler III" "H.H.H. Heavy Haulage"
		ship "Hauler III" "H.H.H. Hroar's Heavy Hitter"
		ship "Hauler II" "H.H.H. First Last Chance"
		ship "Hauler II" "H.H.H. Lucky Hauler"
		ship "Mule" "H.H.H. Herald of Vermilion"

> 		ship "Boxwing" "H.H.H. Dirt-In-My-Eye." 

Reviewer 7: ```suggestion
		ship "Boxwing" "H.H.H. Dirt-In-My-Eye"
```

			`	Light reflects off the tears that cling to Timothy's eyes as he tells the assembled crowd that, for him, that was your greatest moment, your most heroic action, that nothing you've done, or ever will do, compares to that simple act of mercy. And that this, to him, alone, is what makes you great.`
			`	From here his story shifts to your selfless offer to help save Rand, even though they could promise no reward, of your stunning victory over Greenrock, and your daring liberation of the fleet on the ground. He ends with you triumphant, and Rand saved.`
			`	The assembled guests stand, clap and cheer. They look at you, at your seat of honor, and raise their glasses to you. Eventually as the cheers and hoots die down, the Governor declares you a Hero of Rand, and thanks you for your service.`
			choice
				`	(Say a few words.)`
				`	(Mingle with the crowd.)`
					goto mingle
				`	(Skip the party.)`
					goto skip

			`	You walk up to the podium, past the governor and turn to address the assembled guests.`
			choice
				`	(Humbly thank them.)`
					goto thankthem
				`	(Tell them it is well deserved.)`
					goto deserved
				`	(Dress them down for being capitalist pigs.)`

> 			`	You address the crowd. You compliment them on how well dressed they all are, how lavish this mansion is, how wealthy they all appear. They clap, and laugh at your apt observations. But, then you talk about how poor, dirty, and shabby things are outside of this mansion. With its fake gravity, and fake greenery. You declare you want no praise from them, no flattery, and no accolades. There's a million Heroes of Rand, outside these baroque walls, toiling away in dirty, dangerous mines. All for what? So that these company-bosses and fat-cat politicians can pat each other on the back and live in obscene luxury in the middle of a squalid world? You rail, you rant, you break them down, and you turn the crowd against you. Angry people are shouting back at you now, boos and hisses are rising up from all sides of the room. And yet, you see a glimmer of respect in the eyes of servants. The waiters, the attendants, the assistants, they clench their fists and nod as your speech goes on. They hear you. They agree with you. A plate crashes into the podium. Another slams into the pillar nearby you. The crowd bulging toward you. The esteemed guests with murder in their eyes.`

Reviewer 7: ```suggestion
			`	You address the crowd. You compliment them on how well dressed they all are, how lavish this mansion is, how wealthy they all appear. They clap, and laugh at your apt observations. But, then you talk about how poor, dirty, and shabby things are outside of this mansion, with its fake gravity, and fake greenery. You declare you want no praise from them, no flattery, and no accolades. There are a million Heroes of Rand, outside these baroque walls, toiling away in dirty, dangerous mines. All for what? So that these company-bosses and fat-cat politicians can pat each other on the back and live in obscene luxury in the middle of a squalid world? You rail, you rant, you break them down, and you turn the crowd against you. Angry people are shouting back at you now, boos and hisses are rising up from all sides of the room. And yet, you see a glimmer of respect in the eyes of servants. The waiters, the attendants, the assistants, they clench their fists and nod as your speech goes on. They hear you. They agree with you. A plate crashes into the podium. Another slams into the pillar nearby you. The crowd bulging toward you, esteemed guests with murder in their eyes.`
```

				set "med1"
			`	There's numerous shelves filled with all kinds of drugs, and it takes you a few minutes to find the detoxification medicine. By then Bertha is at your side.`
			`	"Foul winds are known to kick up toxic dust in the atmosphere, and if you plan on staying here long you're likely going to need to take a few of these." She looks at you curiously, and then adds, "However, you may find them less useful on other worlds."`
			`	"Still, if you think you could use them I'd sell them to you for 10,000 credits," she adds helpfully.`
			choice
				`	(Buy Dancer's medicine for Tim.)`
					goto getmed
				`	(Pass.)`

			`	Bertha nods. "You have the windless look of a tourist. Best to leave these for those who are going to spend some real time on Dancer."`
				goto danceroptions

			label med2
			action
				set "med2"
			`	The shelves of medication are just where you left them. You can grab a kit of Dancer detoxification drugs for 10,000 credits.`
			choice
				`	(Buy Dancer's medicine for Tim.)`
					goto getmed
				`	(Pass.)`

			`	You leave the drugs where they are and return to looking through the clearance house.`
				goto danceroptions
			
			label getmed
			action

> 				"timothy radrickson medicine" = 1

Reviewer 3: `==` checks for whether a variable is equal to the comparison value; `=` sets the variable to that value, which is what should be happening here.

It's the same as writing:
```
action
  set "timothy radrickson medicine"
```

			`	"Safe travels, privateer," he says as you leave his stall.`
				goto greenoptions

			label getgear3
			action
				set "suit2"
				"timothy radrickson suit" = 1
				payment -100000
			`	The proprietor quickly rings you up as if worried you'll change your mind at any moment, and gleefully takes your <payment>.`
			`	"Safe travels, privateer," he says as you leave his stall.`
				goto greenoptions

			label getgear4
			action
				set "suit2"
				"timothy radrickson suit" = 1
				payment -250000
			`	The proprietor quickly rings you up as if worried you'll change your mind at any moment, and gleefully takes your <payment>.`
			`	"Safe travels, privateer," he says as you leave his stall.`
				goto greenoptions

			label med1
			action
				set "med1"
			`	There are many stalls loudly advertising all kinds of illegal drugs, and it takes some work to sift through them until you come across one selling legal drugs as well. The man running it is sallow and pock-marked, but his eyes are keen. He almost hisses as he greets you.`
			`	"Whatever you need, wherever you're going. I have you covered."`
			choice

> 				`	(Ask about detoxification medicine for Oblivion.)`

Author: I don't think adding an article (a) there makes more sense. 

It doesn't feel natural. "I am looking to buy cancer medicine", "I am looking to buy a cancer medicine." Adding that a just doesn't sound... right.

		random < 25
		has "event: timothy radrickson forever"
	on offer
		conversation
			`A group of monks passes you by in the spaceport, and one stops to look at you curiously.`
			`	"Are you Captain <last>?" They ask.`
			choice
				`	"Why?"`
					goto why
				`	"Yes."`
					goto why
				`	"None of your business."`

			`	"Of course," the monk says. He drops his head and rejoins his brothers.`
				decline

			label why
			`	The monk lowers his head in a sign of respect. "Although we spend most our time mediating and healing the sick, we have heard much of your exploits. You are an agent of change, and worthy of consideration."`
			`	"I thank you for giving us much to ponder on. I also have one small request; there is one within our hospital who is a great admirer of yours. It would mean much if you would be willing to meet him."`
			choice
				`	"I'd be honored to meet him."`
					goto whyyes
				`	"Why not. I've got nothing to better to do."`
					goto whyyes
				`	"I don't have time for fans."`

			`	"Of course," the monk says, "you are a busy person." He returns to his brothers.`
				decline

			label whyyes
			`	The monk beams. "Wonderful. It's not far from here."`

> 			`	You take the spaceport train with him to a nearby hospital, beneath the impressive jungled mountains of New Tibet. In contrast with the other worlds of the Dirt Belt, this hospital is new, clean, and filled with a slew of monks, doctors, and patients. Your escort leads wildly past crowded bedsides and through diverse rooms until, at last, you arrive at a respiration and burns unit situated deep in the hospital's guts.`

Author: In this instance finding yourself deep in a building's 'guts' is a phrase that has some use. Deep in the a building's 'stomach' is not a phrase that is normally used.

			label deserved
			`	You launch into a speech about how you alone are special. How most people are cold, and empty robots, but that a fire burns inside your breast. That you are the linchpin that the entire galaxy hinges on, and that every human from here to Farpoint owes their future to you. The crowd's excitement wanes and a smattering of lackluster applause punctuates the end of your speech.`

			label skip

> 			`	You excuse your self from the table and duck into the first empty room you find. You sit in one of the comfortable chairs you notice a large wooden desk covered in screens and ledgers. From behind the walls you can still hear the muffled sounds of laughter, speeches, and general chatter. You sit for a while in dark meditative silence. `

Reviewer 2: ```suggestion
			`	You excuse yourself from the table and duck into the first empty room you find. You sit in one of the comfortable chairs; you notice a large wooden desk covered in screens and ledgers. You can still hear the muffled sounds of laughter, speeches, and general chatter from behind the walls. You sit for a while in a dark meditative silence. `
```

> 			`	After a while there's a gentle knock on the door and Timothy pushes his way into the room.`

Reviewer 2: ```suggestion
			`	After a while, a gentle knock on the door and Timothy pushes his way into the room.`
```

> 			`	"I see you've found my office, Captain <last>. I'd like to apologize, I was hoping you'd appreciate the party, but I realize it's not exactly everyone's scene. There's more then a few vipers in that crowd and there are things I've had to do for them I am not proud of..."`

Reviewer 2: ```suggestion
			`	"I see you've found my office, Captain <last>. I'd like to apologize. I was hoping you'd appreciate the party, but, I realize it's not exactly everyone's scene. There's more then a few vipers in that crowd and there are things I've had to do for them I am not proud of..."`
```

> 			`	He trails off, "But still I am glad you're here. Even tucked away in my office. I wanted to let you know that there is cash associated with tonight's reward. You will be given 1 million credits and a daily stipend of 200 credits, as a gift from Rand, and more specifically, from me. `

Reviewer 2: ```suggestion
			`	He trails off, "But still, I'm glad you're here. Even tucked away in my office. I wanted to let you know that there is credits associated with tonight's prize. You will be given 1 million credits and a daily stipend of 200 credits, as a gift from Rand, and more specifically, from me. `
```

			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`

mission "Timothy Radrickson 1: Stowaway"
	invisible
	landing
	source
		near "Zeta Aquilae" 3 10
		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"

> 	destination "Rand"

Author: As it was written for Rand- Rand currently works the best. If this is a real sticking point I think I can make Arabia fit. I'd obviously have to remove all the references to gravity generators and stooped workers.

> 	destination "Rand"

Reviewer 5: I think changing New Arabia is a bit out of the scope of this PR, but if you'd like to see the description changed feel free to open up another PR.
I agree with SpearDane that changing the setting to New Arabia or some other planet would require a massive rewrite of the mission that in my opinion isn't worth it.

> 	destination "Rand"

Reviewer 8: If i do end up addressing this, it'll probably be in a follow up PR.

			action
				fail
			`	The feeble man screams as you drag him out of the crate and toward the airlock. He ineptly squirms as you shove him past the bulkhead seals. He falls to his knees shuddering, and he scrunches up with terror. You close the airlock, silencing his shrill screams, and hit the button to open the outside airlock. The man is promptly sucked out into cold, cruel space and crystallizes before becoming this system's newest comet.`
				decline

			label port
			action
				fail
				set "timothy radrickson: port"
			`	"Thank you," he whispers as tears squeeze out of his eyes. "Thank you."`
			`	You get the man cleaned up and settled, and then go back to running your ship.`
				decline

			label somewhere
			`	Timothy looks at you, his eyes wide and full of tears. "Rand," he whispers. "Rand. I can make good money there. I can turn my life around. B-b-but I'd be okay with anywhere. You d-d-don't have to.."`
			`	You cut him off. "It's fine, Timothy. I'll take you to Rand."`
			`	Timothy breaks down in tears. "Thank you. Thank you. Thank you..." he mumbles over and over again.`
			`	You help him out of the crate and get him cleaned up.`
				accept

	on complete

> 		log `Discovered Timothy, a stowaway, in my ship, and decided to take him where he needed to go. Never learned what Timothy had been fleeing, who he had been, or what he had done that lead him to such dire straits.`

Author: Okay. I have done this.

			`For days there's been an odd lingering scent filtering in from your pressurized cargo. The smell has finally permeated your command deck, and something had to be done. A ship-wide search has led you to a unopened foodstuff crate. Although the crate was unopened, the seal on it appeared tampered with and the conjecture was that the food had gone bad.`
			`	Upon opening it you discovered that was in fact, not the case. Instead you discovered the decaying body of a dead and forgotten stowaway; the entire crate promptly found its way to your airlock. Despite the diligent work of your ship's air filters, you can't quite rid the ship of the smell.`

mission "Timothy Radrickson 2: Interlude"
	landing
	to offer
		has "Timothy Radrickson 1: Stowaway: failed"
		has "timothy radrickson: port"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	on offer
		conversation
			`Blinking at the light, Timothy shuffles toward the exit. "Th-th-thank you. Thank you so much," Timothy mumbles.`
			`	Timothy opens the hatch and looks back at you. "You s-s-saved my life. I won't forget. I won't."`
			`	With that, Timothy exits your ship and begins his new life.`

> mission "Timothy Radrickson 2: Good Interlude"

Author: Doesn't it already have: 
government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"

Would I just need to remove Neutral and Independent to ensure it happens in human space?

> mission "Timothy Radrickson 2: Good Interlude"

Reviewer 8: It doesn't have a `source` node.

> mission "Timothy Radrickson 2: Good Interlude"

Author: Ah, added one.

			`	The pointing stick falls on Shaula next. "The best place for a fake ID would be the 'independent' world of Greenrock." She taps her chin before continuing, "It would be tight though getting to all three of these worlds and back to the governor in time. Any of these worlds may also have the other equipment you need, but the quality likely wouldn't be as good."`
			`	She turns back to you. "I am sorry we couldn't be more help, but hopefully it's enough."`
			`	After putting the pointing stick back in its place, Wynnafrith steps through the door, indicating for you to follow her.`
			choice
				`	(Follow her.)`

			`	Once again in the larger cavern she turns and points at the guard. "Grab our explosive and carefully accompany Captain <last> with it back to his ship."`
			`	The guard frowns. "But it's our only one..."`
			`	"Now!" She cuts him off, and watches as he sullenly slumps away.`
			`	Without looking your way she continues, "It was a pleasure to see you again, Captain <last>, but it's time you were leaving. The governor needs you, and my own glorious revolution needs some attention."`

> 			`	You leave the dilapidated mines with a bomb, a pistol, and a plan.`

Reviewer 5: Perhaps have a log entry here regarding this mission?

Author: Done.

				goto end

			label "well prepared"
			`	As he inspects the suit, drugs, and ID you can see his demeanor shift. He looks less weak, less sickly, and more confident.`
			`	Finally, he straightens back up. "It's perfect." He eyes the gear with awe and shakes his head. "Don't know how you did it. But with this I can't fail."`

			label end
			`	Timothy turns from you and uses the dolly to load the gear into one of the trucks in the warehouse. Once done he turns back to you, his eyes wide and rimmed with bloody tears. "Always at my lowest." He swallows a cough, and then continues. "It's you who saves me. When I am at my lowest. It's you."`
			`	Timothy looks like he wants to hug you, but he just nods. He hits a button on the wall and the garage door opens, exposing the warehouse to the night air.`
			`	"Captain," he says in way of a farewell, and then he climbs aboard the truck and drives off.`

mission "Timothy Radrickson 6: News of Death"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		attributes "urban"
	to offer
		has "Timothy Radrickson 5e: Back to Oblivion: offered"
		random < 15
	on offer

> 		log "Timothy is dead. Or at least that is what is being reported in the news. But maybe you did enough to help him escape."

Reviewer 5: ```suggestion
		log "Timothy is dead - or at least that is what's being reported in the news. Perhaps the plot to break him free really did work."
```

				`	(Shake his hand and say goodbye.)`
					goto shake
				`	(Give him a hug and say goodbye.)`

			action
				clear "offers"
			`	You lean in and give him a warm, earnest bear hug. He wraps his arms around you and squeezes you back. When he finally breaks free you see a tear his eyes are blurry with tears. He nods to you once, and then walks back into the monastery. You begin your slow descent down the mountain, and back to civilization.`
				decline

			label shake
			action
				clear "offers"
			`	You shake his hand; he swallows and clears his throat as he tries to keep himself from crying, nods to you one last time, and then walks back into the monastery. You begin your slow descent down the mountain, and back to civilization.`
				decline
		
			label offer
			action
				set "offers"
			`	Timothy's eyes go wide. "No, you couldn't. It's yours. You've given me too much already."`
			choice
				`	(Tell him about what Wynnafrith said, and how he is the real Hero of Rand.)`
					goto offer2
				`	(Tell him you don't need the money, but he does.)`
					goto offer2
				`	(Agree with him and rescind your offer.)`
					goto goodbyes
			
			label offer2
			action

> 				log "Timothy is alive and well on New Tibet. Living a simple, but happy life as a monk."

Reviewer 5: ```suggestion
				log "Found Timothy alive and well on New Tibet, living a simple, but happy life as a monk."
```

			`	He leads you back to the train station, and from there you take brief journey back to New Tibet's spaceport.`
				decline

			label shake
			action
				clear "offers"
			`	You lean in to shake his good hand. He swallows and clears his throat, trying to keep himself from crying.`
			`	You leave him bedridden in the hospital, and to your surprise find your guide waiting for you outside the room. "His is a tough case. Perhaps one day he will recover enough to leave this place." The monk pauses, thoughtfully. "Or perhaps not."`
			`	He leads you back to the train station, and from there you take brief journey back to New Tibet's spaceport.`
				decline
		
			label offer
			action
				set "offers"
			`	Timothy's eyes go wide. "No, you couldn't. It's yours. You've given me too much already."`
			choice
				`	(Tell him about what Wynnafrith said, and how he is the real Hero of Rand.)`
					goto offer2
				`	(Tell him you don't need the money, but he does.)`
					goto offer2
				`	(Agree with him and rescind your offer.)`
					goto goodbyes
			
			label offer2
			action

> 				log "Timothy is alive, but not in great shape. He thankful that you saved him, but maybe you could have done for him."

Reviewer 5: ```suggestion
				log "Found Timothy alive, but not in great shape. He is thankful for the rescue nonetheless. Can't shake the feeling that there was more that I could have done for him."
```

			`	With that heavy thought on your mind you notice a sign which promises to lead you to the local graveyard.`
			choice
				`	(Head to the graveyard.)`
					goto graveyard
				`	(Head back to your ship.)`

			`	You see no point in wasting your time at this place's graveyard, so you head back to your ship.`
				decline
			
			label graveyard
			`	Oblivion's evening sky is lit up by unnumbered rigs and natural gas burners that dot the region. These red fires turn the normally dark blue atmosphere a shade of smoldering purple.`
			`	It's beneath that purple hue you see a countless expanse of small flat gravestones uniformly placed in Oblivion's sterile dirt. Despite the number of headstones there is no one else here, and only a few graves show any evidence of being visited.`
			`	It seems as though those buried here are mostly forgotten.`
			choice
				`	(Search for Timothy Radrickson's grave.)`
					goto search
				`	(Leave.)`

			`	You've seen all you wanted to see of this place, and you leave this unloved graveyard to its forlorn dead.`
				decline

			label search

> 			log "Timothy is dead. It seems you failed him."

Reviewer 7: ```suggestion
			action
				log "Timothy is dead. It seems you failed him."
```

Reviewer 5: ```suggestion
			action
				log "Timothy is dead. It seems the plan to break him free failed."
```

			action
				fail
			`	The feeble man screams as you drag him out of the crate and toward the airlock. He ineptly squirms as you shove him past the bulkhead seals. He falls to his knees shuddering, and he scrunches up with terror. You close the airlock, silencing his shrill screams, and hit the button to open the outside airlock. The man is promptly sucked out into cold, cruel space and crystallizes before becoming this system's newest comet.`
				decline

			label port
			action
				fail
				set "timothy radrickson: port"
			`	"Thank you," he whispers as tears squeeze out of his eyes. "Thank you."`
			`	You get the man cleaned up and settled, and then go back to running your ship.`
				decline

			label somewhere
			`	Timothy looks at you, his eyes wide and full of tears. "Rand," he whispers. "Rand. I can make good money there. I can turn my life around. B-b-but I'd be okay with anywhere. You d-d-don't have to.."`
			`	You cut him off. "It's fine, Timothy. I'll take you to Rand."`
			`	Timothy breaks down in tears. "Thank you. Thank you. Thank you..." he mumbles over and over again.`
			`	You help him out of the crate and get him cleaned up.`
				accept

	on complete

> 		log "People" "Timothy Radickson" `A stowaway I found on my ship. He requested a trip to Rand to start a new life, promising to pay back the money he owes..`

Reviewer 5: ```suggestion
		log "People" "Timothy Radickson" `A stowaway I found on my ship. He requested a trip to Rand to start a new life, promising to pay back the money he owes.`
```

				goto end

			label "well prepared"
			`	As he inspects the suit, drugs, and ID you can see his demeanor shift. He looks less weak, less sickly, and more confident.`
			`	Finally, he straightens back up. "It's perfect." He eyes the gear with awe and shakes his head. "Don't know how you did it. But with this I can't fail."`

			label end
			`	Timothy turns from you and uses the dolly to load the gear into one of the trucks in the warehouse. Once done he turns back to you, his eyes wide and rimmed with bloody tears. "Always at my lowest." He swallows a cough, and then continues. "It's you who saves me. When I am at my lowest. It's you."`
			`	Timothy looks like he wants to hug you, but he just nods. He hits a button on the wall and the garage door opens, exposing the warehouse to the night air.`
			`	"Captain," he says in way of a farewell, and then he climbs aboard the truck and drives off.`

mission "Timothy Radrickson 6: News of Death"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		attributes "urban"
	to offer
		has "Timothy Radrickson 5e: Back to Oblivion: offered"
		random < 15
	on offer

> 		log "People" "Timothy Radickson" `A stowaway I found on my ship long ago. Who became the Governor of Rand, and then fell from grace. I tried to help him escape from a slow death on Oblivion. Time will tell if I was successful or not.`

Reviewer 5: ```suggestion
		log "People" "Timothy Radickson" `Tried to help Timothy escape from his prison sentence there. Time will tell if the plan was successful or not.`
```

		log "Timothy is dead - or at least that is what's being reported in the news. Perhaps the plot to break him free really did work."
		event "timothy radrickson forever"
		conversation
			`A spaceport is a symphony of smells, sounds, and images in constant motion. Somewhere in the din, a news report is playing, and a familiar name leaps out at you. Your attention piqued, you turn to watch.`
			`	"News out of Oblivion today. A minor explosion has the claimed the life of Timothy Radrickson, former disgraced Governor of Rand. Sentenced to prison-labor on Oblivion, a hard but fair justice for a man who..."`
			`	The news broadcast continues to blare, but you aren't able to follow any more than that.`
				decline

event "timothy radrickson forever"

mission "Timothy Radrickson 7: Monk"

> 	source "New Tibet"

Reviewer 7: Perhaps make this `minor` so that it doesn't offer during FWR.
```suggestion
	minor
	source "New Tibet"
```

				`	(Shake his hand and say goodbye.)`
					goto shake
				`	(Give him a hug and say goodbye.)`

			action
				clear "offers"
			`	You lean in and give him a warm, earnest bear hug. He wraps his arms around you and squeezes you back. When he finally breaks free you see a tear his eyes are blurry with tears. He nods to you once, and then walks back into the monastery. You begin your slow descent down the mountain, and back to civilization.`
				decline

			label shake
			action
				clear "offers"
			`	You shake his hand; he swallows and clears his throat as he tries to keep himself from crying, nods to you one last time, and then walks back into the monastery. You begin your slow descent down the mountain, and back to civilization.`
				decline
		
			label offer
			action
				set "offers"
			`	Timothy's eyes go wide. "No, you couldn't. It's yours. You've given me too much already."`
			choice
				`	(Tell him about what Wynnafrith said, and how he is the real Hero of Rand.)`
					goto offer2
				`	(Tell him you don't need the money, but he does.)`
					goto offer2
				`	(Agree with him and rescind your offer.)`
					goto goodbyes
			
			label offer2
			action

> 				log "Found Timothy alive and well on New Tibet, living a simple, but happy life as a monk."

Reviewer 5: Perhaps add a person log here about Timmy's ultimate fate.

Author: Done.

			`	He leads you back to the train station, and from there you take brief journey back to New Tibet's spaceport.`
				decline

			label shake
			action
				clear "offers"
			`	You lean in to shake his good hand. He swallows and clears his throat, trying to keep himself from crying.`
			`	You leave him bedridden in the hospital, and to your surprise find your guide waiting for you outside the room. "His is a tough case. Perhaps one day he will recover enough to leave this place." The monk pauses, thoughtfully. "Or perhaps not."`
			`	He leads you back to the train station, and from there you take brief journey back to New Tibet's spaceport.`
				decline
		
			label offer
			action
				set "offers"
			`	Timothy's eyes go wide. "No, you couldn't. It's yours. You've given me too much already."`
			choice
				`	(Tell him about what Wynnafrith said, and how he is the real Hero of Rand.)`
					goto offer2
				`	(Tell him you don't need the money, but he does.)`
					goto offer2
				`	(Agree with him and rescind your offer.)`
					goto goodbyes
			
			label offer2
			action

> 				log "Found Timothy alive, but not in great shape. He is thankful for the rescue nonetheless. Can't shake the feeling that there was more that I could have done for him."

Reviewer 5: Perhaps add a person log here about Timmy's ultimate fate.

Author: Done.

			`	With that heavy thought on your mind you notice a sign which promises to lead you to the local graveyard.`
			choice
				`	(Head to the graveyard.)`
					goto graveyard
				`	(Head back to your ship.)`

			`	You see no point in wasting your time at this place's graveyard, so you head back to your ship.`
				decline
			
			label graveyard
			`	Oblivion's evening sky is lit up by unnumbered rigs and natural gas burners that dot the region. These red fires turn the normally dark blue atmosphere a shade of smoldering purple.`
			`	It's beneath that purple hue you see a countless expanse of small flat gravestones uniformly placed in Oblivion's sterile dirt. Despite the number of headstones there is no one else here, and only a few graves show any evidence of being visited.`
			`	It seems as though those buried here are mostly forgotten.`
			choice
				`	(Search for Timothy Radrickson's grave.)`
					goto search
				`	(Leave.)`

			`	You've seen all you wanted to see of this place, and you leave this unloved graveyard to its forlorn dead.`
				decline

			label search
			action

> 				log "Timothy is dead. It seems the plan to break him free failed."

Reviewer 5: Perhaps add a person log here about Timmy's ultimate fate.

Author: Done.

				goto end

			label "well prepared"
			`	As he inspects the suit, drugs, and ID you can see his demeanor shift. He looks less weak, less sickly, and more confident.`
			`	Finally, he straightens back up. "It's perfect." He eyes the gear with awe and shakes his head. "Don't know how you did it. But with this I can't fail."`

			label end
			`	Timothy turns from you and uses the dolly to load the gear into one of the trucks in the warehouse. Once done he turns back to you, his eyes wide and rimmed with bloody tears. "Always at my lowest." He swallows a cough, and then continues. "It's you who saves me. When I am at my lowest. It's you."`
			`	Timothy looks like he wants to hug you, but he just nods. He hits a button on the wall and the garage door opens, exposing the warehouse to the night air.`
			`	"Captain," he says in way of a farewell, and then he climbs aboard the truck and drives off.`

mission "Timothy Radrickson 6: News of Death"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		attributes "urban"
	to offer
		has "Timothy Radrickson 5e: Back to Oblivion: offered"
		random < 15
	on offer

> 		log "People" "Timothy Radickson" `Tried to help Timothy escape from his prison sentence there. Time will tell if the plan was successful or not.`

Reviewer 5: ```suggestion
		log "People" "Timothy Radickson" `Tried to help Timothy escape from his prison sentence there.`
```

			action
				clear "offers"
			`	You lean in and give him a warm, earnest bear hug. He wraps his arms around you and squeezes you back. When he finally breaks free you see a tear his eyes are blurry with tears. He nods to you once, and then walks back into the monastery. You begin your slow descent down the mountain, and back to civilization.`
				decline

			label shake
			action
				clear "offers"
			`	You shake his hand; he swallows and clears his throat as he tries to keep himself from crying, nods to you one last time, and then walks back into the monastery. You begin your slow descent down the mountain, and back to civilization.`
				decline
		
			label offer
			action
				set "offers"
			`	Timothy's eyes go wide. "No, you couldn't. It's yours. You've given me too much already."`
			choice
				`	(Tell him about what Wynnafrith said, and how he is the real Hero of Rand.)`
					goto offer2
				`	(Tell him you don't need the money, but he does.)`
					goto offer2
				`	(Agree with him and rescind your offer.)`
					goto goodbyes
			
			label offer2
			action
				log "Found Timothy alive and well on New Tibet, living a simple, but happy life as a monk."

> 				log "Timotyh Radrickson" `Timothy Radrickson survived his faked death on Oblivion and has escaped to New Tibet, where, thanks to my help he now lives as simple monk. He seems happy.`

Reviewer 7: ```suggestion
				log "Timothy Radrickson" `Timothy Radrickson survived his faked death on Oblivion and has escaped to New Tibet, where, thanks to my help, he now lives as simple monk. He seems happy.`
```

			`	Light reflects off the tears that cling to Timothy's eyes as he tells the assembled crowd that, for him, that was your greatest moment, your most heroic action, that nothing you've done, or ever will do, compares to that simple act of mercy. And that this, to him, alone, is what makes you great.`
			`	From here his story shifts to your selfless offer to help save Rand, even though they could promise no reward, of your stunning victory over Greenrock, and your daring liberation of the fleet on the ground. He ends with you triumphant, and Rand saved.`
			`	The assembled guests stand, clap and cheer. They look at you, at your seat of honor, and raise their glasses to you. Eventually as the cheers and hoots die down, the Governor declares you a Hero of Rand, and thanks you for your service.`
			choice
				`	(Say a few words.)`
				`	(Mingle with the crowd.)`
					goto mingle
				`	(Skip the party.)`
					goto skip

			`	You walk up to the podium, past the governor and turn to address the assembled guests.`
			choice
				`	(Humbly thank them.)`
					goto thankthem
				`	(Tell them it is well deserved.)`
					goto deserved
				`	(Dress them down for being capitalist pigs.)`

> 			`	You address the crowd. You compliment them on how well dressed they all are, how lavish this mansion is, how wealthy they all appear. They clap, and laugh at your apt observations. But, then you talk about how poor, dirty, and shabby things are outside of this mansion, with its fake gravity, and fake greenery. You declare you want no praise from them, no flattery, and no accolades. There are a million Heroes of Rand, outside these baroque walls, toiling away in dirty, dangerous mines. All for what? So that these company-bosses and fat-cat politicians can pat each other on the back and live in obscene luxury in the middle of a squalid world? You rail, you rant, you break them down, and you turn the crowd against you. Angry people are shouting back at you now, boos and hisses are rising up from all sides of the room. And yet, you see a glimmer of respect in the eyes of servants. The waiters, the attendants, the assistants, they clench their fists and nod as your speech goes on. They hear you. They agree with you. A plate crashes into the podium. Another slams into the pillar nearby you. The crowd bulging toward you, esteemed guests with murder in their eyes.`

Author: I have re-written it as you requested.

			`	Before you have time to act, Timothy and Wynnafrith have whisked you away from the angry crowd of the world's elites. Timothy, unhappy with your words, but still believing he owes you a debt, helps you into an unmarked vehicle that quickly takes you away from the party, and back toward the spaceport and your ship. Timothy's final words to you were that you should get off-world as quickly as you can, and that he is sorry he invited you here.`
				decline

			label thankthem
			`	You thank your host, and the assembled notables for their gracious fanfare. You note, however, that you are just one person. And you've done nothing that any one else in your shoes wouldn't have done. Kindness is contagious, and everyone is a hero in some way. You hope that the takeaway here is not that you are special, but instead that everyone should just do their part. The guests clap as you step away from the podium.`
				goto mingle

			label deserved

> 			`	You launch into a speech about how you alone are special. How most people are cold and empty robots, but that a fire burns inside your breast. That you are the linchpin that the entire galaxy hinges on, and that every human from here to Farpoint owes their future to you. The crowd's excitement wanes and a smattering of lackluster applause punctuates the end of your speech.`

Author: I made the change to the earlier line you requested. Instead of a mob, they just lose interest.

			`	She takes your <payment> and hands you the old ID, her eyes lingering on you.`
				goto danceroptions

			label end
			action
				clear "med1"
				clear "med2"
				clear "suit1"
				clear "suit2"
				clear "id1"
				clear "id2"
			`	Bertha gives you a friendly nod as you make your way back outside into Dancer's foreboding weather, and the rest of the spaceport.`
				decline
			

mission "Timothy Radrickson 5d: Greenrock"
	name "Supplies on <planet>"
	description "Tim's old secretary Wynnafrith recommended that <destination> would be a good place to find a fake ID for Tim."
	landing
	source Rand
	destination Greenrock
	to offer
		has "Timothy Radrickson 5a: Timshank Redemption: active"
	on offer
	on complete
		conversation
			`You feel the pressure of many unseen eyes watching you as you leave the spaceport and enter into the sprawling stalls of Greenrock's rowdy emporium, where everything and everyone is for sale.`

> 			`	Greenrock looks a little more civil than when you were racing around saving captured freighter fleets, but only just a little. There is a wildness here and violence simmers just beneath the surface of every interaction. Luckily, no one seems to remember or care about the fleet you destroyed.`

Author: Uuuh I haven't looked at this for a while. But I think you have to save the freighters earlier in the mission chain to get to where Tim gets sent to the penal colony. 

I don't think you can decline to save the fleet, but still save Tim later.

> 			`	Greenrock looks a little more civil than when you were racing around saving captured freighter fleets, but only just a little. There is a wildness here and violence simmers just beneath the surface of every interaction. Luckily, no one seems to remember or care about the fleet you destroyed.`

Author: nevermind the above.. I looked it up. It appears you can decline to help, but still get tim imprisoned event. That's interesting. Something Bene must have added. Hmm.

> 			`	Greenrock looks a little more civil than when you were racing around saving captured freighter fleets, but only just a little. There is a wildness here and violence simmers just beneath the surface of every interaction. Luckily, no one seems to remember or care about the fleet you destroyed.`

Author: Yeah, sure. If you notice anything like that it'd help. 

Also, I can't reply to correction about the spaceport below for some reason, but I think I still like the way it was originally written.

			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`

mission "Timothy Radrickson 1: Stowaway"
	invisible
	landing
	source
		near "Zeta Aquilae" 3 10
		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
	destination "Rand"
	passengers 1
	to offer
		random < 10
	on enter

	# You will see this conversation after the first jump you make.
	on enter
		conversation
			`Upon completing your jump, one of your internal sensors begins to beep. You flick off the siren and check the diagnostic report: apparently there is a non-negligible heat source detected in the food storage of your ship.`
			`	As problems go, it is not a particularly alarming one: there are plenty of reasons there may be a heat spike in storage, and most of them would not require the captain of the ship to investigate.`
			`	It is unusual, however.`
			choice
				`	(Investigate.)`
					goto check
				`	(Ignore the sensor.)`

			action

> 				fail

Reviewer 3: I believe the fail is necessary because `Timothy Radrickson 2: mummy` only fires if you fail this mission (and I placed the failures in places where Timothy would die to your inaction). Moreover it immediately removes the mission from your active missions list while `decline` doesn't actually do anything but end the conversation in this instance. (You have technically already accepted `Timothy Radrickson 1: Stowaway` when you see this conversation, it's not an `on offer` conversation.)

I had thought in the past to streamline it and simply remove that second option, but I have no time to at the moment.
```
mission "Timothy Radrickson 2: mummy"
[...]
	to offer
		has "Timothy Radrickson 1: Stowaway: failed"
```

				event "timothy radrickson: mummy" 41
			`	It's almost certainly nothing, and you have more important things to do than chase down every little sensor glitch on your ship.`
				decline

			label check
			`	After a short search, and the rearranging of quite a few deceptively heavy boxes, you are able to track the heat signature to a foodstuff crate you recently took on. It has yet to be opened, but you can tell its seal has already been tampered with.`
			choice
				`	(Open the crate.)`
					goto crate
				`	(Dump the crate out the airlock.)`

			action

> 				fail

Reviewer 3: See previous comment

	description "The governor of <planet> has invited you to speak with them."
	to offer
		has "event: timothy radrickson: governor"
		has "main plot completed"
	destination "Rand"
	landing
	on offer
		conversation
			`An indicator light is blinking on your main panel. There is a cached message waiting for you to play.`
			choice
				`	(Play the message.)`
			`	A young woman in very professional clothing appears on the screen. "Captain <last>, I am pleased to inform you that the Governor of Rand requests your presence. Please come to Rand at your earliest convenience.`

	on complete
		conversation
			`Upon landing you are met by a squat, heavy-set woman in her late forties. She reaches out a calloused hand to shake yours. "Welcome to Rand, Captain <last>. It is a pleasure, and honor, to have someone of your status here on our little world. I am Regina, the mayor of this town, and one of the major company-bosses here. I've been instructed to escort you to the Governor's mansion. I have a vehicle waiting for you just outside the spaceport. Meet me there when you're ready."`

mission "Timothy Radrickson 3a: Rescue the Convoy"
	name "Rescue <origin> convoy"

> 	description "<origin>'s largest ever convoy was intercepted on its way to Tarazed. Merchants report that a large pirate fleet escorting a freighter convoy passed by Lesath, and it looks like they went to <destination>. Governor Tim has asked you to try and save it before the pirates can plunder it completely."

Reviewer 9: Did you mean to resolve this one -- it doesn't look like it's been addressed? Maybe its fine to exceed the max space by the bit here, though I think it makes sense to get people's opinions on this since it's outside the bounds of normal missions.

				launch

			label surprise
			action
				set "big greenrock pirate fleet"
			`	You shunt your thrusters to full speed and fly into the valley, straight for the largest cluster of parked ships outside the blast doors.`
			`	That is, until your sensors run hot with a multitude of approaching enemy warnings and missile locks. Without the protection of the hills that were hiding your ship, it looks like the pirates' early warning system detected you before you could get into position to attack. You're forced to turn back as an entire fleet of marauders fire up their engines and chase you into space.`
				launch

	on visit
		dialog `There are still marauder ships flying overhead. Take off and defeat them before finding the fleet.`

	npc kill
		to spawn
			has "big greenrock pirate fleet"
		government "Pirate"
		personality launching nemesis target plunders
		fleet
			names "pirate"
			variant
				"Marauder Falcon (Engines)"
				"Falcon (Javelin)"
				"Marauder Firebird (Weapons)"
				"Firebird"

				"Marauder Raven"
				"Modified Argosy (Gatling)" 2
				"Osprey (Missile)"

				"Hawk (Plasma)"
				"Quicksilver" 2
				"Headhunter"

> 				"Fury (Gatling)" 3

Reviewer 7: Except for marauder should be capitalized?

> 				"Fury (Gatling)" 3

Reviewer 9: Elsewhere they use marauder with lower case elsewhere and the pirate ships aren't all Marauder variants, so I _think_ in this case they mean marauder in the general sense of being a synonym for raider/pirate rather than specifically Marauder ships in the Proper Noun sense.

	destination "Oblivion"
	deadline 1800
	invisible
	on offer
		conversation
			`As you stroll through the spaceport, a name blaring on a nearby news screen catches your attention.`
			choice
				`	(Watch the news.)`
					goto news
				`	(Ignore it.)`
			`	You keep walking. It's probably nothing worth your time.`
				decline
			label news
			`	You turn to check out the screen. The talking heads are discussing some sort of huge scandal and the arrest of a prominent governor.`
			`	"That's right, Ann," the anchor on the screen says. "With 8 counts of tax evasion, racketeering, and various corruption charges, Timothy Radrickson - former Governor of Rand - will likely spend the next 20 years behind bars."`
			`	"Where he's going there will be no bars, Tom," Ann quips. "They plan to send him to a work detail on Oblivion."`
			`	Tom gives that sort of news anchor laugh that's neither committal nor haughty. "In that case, forget what I said about 20 years. Anything more than 5 years on Oblivion is a life sentence."`
			`	Both news anchors laugh, and then segue into a story about highly intelligent ferrets that have learned to sing in German.`
				accept

> 

Reviewer 3: I can't exactly remember the purpose of this because I haven't written ES missions in a while, but it _seems_ to make sense.

> 

Reviewer 9: Yeah, for a bunch of these I'm just playing, getting stuck, and looking at the save file/mission data for what went wrong and fiddling until a fix makes the mission playable. For this one, it showed up as failed immediately and adding in the `to complete` fixed it. Couldn't tell you the why behind it though.

				`	(Leave the trade sector.)`
					goto end

			label med1
			action
				set "med1"
			`	You make your way to the pharmaceutical wing, through crowds of representatives from large conglomerates in the process of wrangling with representatives from various worlds, all needing exotic medicine for their innumerable plagues, blights, pandemics, and the like. There's a booth with a slick salesman selling expensive drugs promising to reverse and protect from the effect of toxic air and radiation, and it just so happens that Oblivion is their demonstration planet.`
			`	"There's no need to fear Oblivion's deleterious effects so long as you take one of these a day!" the salesman says brightly. "Why, you could walk among your condemned workers unmasked, and demonstrate your superior executive prowess!"`
			`	You listen on to the salesman, who reckons he can cut you in on a special deal for 140,000 credits.`
			choice
				`	(Purchase the medicine for Tim.)`
					goto "med 140"
				`	(Haggle for a lower price.)`
					goto haggle
				`	(Pass.)`
			`	That price is too steep. You return to the central atrium of the trade sector to consider your options.`
				goto options
			
			label haggle

> 				`The salesman's eyes narrow and his face suddenly looks vaguely reptilian. "I'm already giving you the best price. The literal shirt off my back!"`

Reviewer 9: ```suggestion
			`The salesman's eyes narrow and his face suddenly looks vaguely reptilian. "I'm already giving you the best price. The literal shirt off my back!"`
```

			choice
				`	(Offer 60,000 for the meds.)`
				`	(Offer 120,000 for the meds.)`
					goto justright
				`	(Agree to pay 140,000 for the meds.)`
					goto "med 140"
				
			`	The drug salesman's eyes flash with hostility before he breaks out in an angry laugh. "Who do you think you are? What do you think, this is a charity? I already have a buyer lined up for more than twice that!"`
			`	He begins packing up his equipment. "I'm charging you full price, 140,000 or nothing. Last chance."`
			choice
				`	(Purchase the medicine for Tim.)`
					goto "med 140"
				`	(Pass.)`
			action
				set "med2"
			`	That price is too high; you'll have to see if you can purchase a less potent and cheaper drug elsewhere. You return to the central atrium of the trade sector to consider your other options.`
				goto options

			label justright

> 				`	The salesman glowers, but nods his head slightly. "Fine! 120,000 credits it is. My family will starve, but at least you'll get your precious drugs." He begins packing them up for you.`

Reviewer 9: ```suggestion
			`	The salesman glowers, but nods his head slightly. "Fine! 120,000 credits it is. My family will starve, but at least you'll get your precious drugs." He begins packing them up for you.`
```

			choice
				`	(Give up and go get something else.)`
					goto options
				`	(Approach the teenagers for help about the ID.)`

			`	The teenagers, two of them boys and one a girl, grin at you like fresh meat as you near them.`
			`	The guy in the middle says, "What'dya need, boss? Wanna get high? Wanna get low? Wanna get out?" Apparently this group is used to working with frustrated castouts from the overly legal spaceport emporium.`
			choice
				`	(Ask them about where you could get a fake ID.)`
					goto hookmeup
				`	(Politely decline and head back to your ship.)`
					goto options
				`	(Threaten to call the police.)`

			`	They back away from you and frown. The middle one lowers his hand to some concealed weapon, but stops himself when he notices your own holstered gun.`
			`	The girl next to him mutters, "Ain't worth it. Let's roll." The kids promptly walk off, leaving you alone in the park.`
			choice
				`	(Head back to the spaceport.)`
					goto options
			
			label hookmeup

> 				`	The three of them look at each other and then give you a carnivorous grin.`
> 				`	"Sure thing," he smirks. "Easy as rain. It'll cost ya 'bout 20,000 cred." They look at you expectantly.`

Reviewer 9: ```suggestion
			`	The three of them look at each other and then give you a carnivorous grin.`
			`	"Sure thing," he smirks. "Easy as rain. It'll cost ya 'bout 20,000 cred." They look at you expectantly.`
```

						not "id2"
				`	(Leave the clearing house.)`
					goto end

			label suit1
			action
				set "suit1"
			`	There are a handful of survival suits on display here. These are durable, hand-crafted suits carefully checked and rated to ensure the highest chance of survival against the fierce uncaring winds of Dancer.`
			`	Bertha notices your attention and walks up to you. "Going anywhere overland on Dancer without one of these beauties is a death wish."`
			`	She winks at you, as if she knows something you don't. "Worry not though, hun. We only sell the highest rated and best-tested suits here. I won't have anyone's death on my conscience - 'sides, it's bad for business."`
			`	She pinches some of the dense microwoven fabric of the nearest suit. "It's not cheap, but neither is your life. I'll sell ya one of these for 300,000 credits."`
			choice
				`	(Purchase a survival suit for Tim.)`
					goto getsuit
				`	(Haggle to try and lower the price.)`
					goto haggle
				`	(Pass.)`

			`	You decide that 300,000 credits is way too much. The woman shrugs and steps away as you turn your attention back to the rest of the store.`
				goto danceroptions
			
			label haggle

> 				`	Bertha gives you a sad smile and shakes her head slowly. "No room here for haggling, hun. These are the best of the best and they are worth every red cent. I am sure you can find bottom bin ones elsewhere, but here you pay for quality."`
> 				`	She folds her arms. "Now, it's 300,000 credits or nothing."`

Reviewer 9: ```suggestion
			`	Bertha gives you a sad smile and shakes her head slowly. "No room here for haggling, hun. These are the best of the best and they are worth every red cent. I am sure you can find bottom bin ones elsewhere, but here you pay for quality."`
			`	She folds her arms. "Now, it's 300,000 credits or nothing."`
```

			`	The drug-dealer says as you go away, "You won't find a better deal than this. I'll be here if you change your mind."`
				goto greenoptions

			label med2
			action
				set "med2"
			`	The drug-dealer's knowing smile is almost a sneer as you return to his stall. He waits to see if you're ready to buy something.`
			choice
				`	(Buy the expired detoxification medicine for Tim.)`
					goto getmed
				`	(Pass.)`

			`	You leave the drug-dealer behind and head back into the emporium.`
				goto greenoptions
			
			label getmed
			action
				set "med2"
				"timothy radrickson	medicine" = 2
				payment -50000
			`	"Trust me, it's still good," the drug-dealer says as he takes your <payment>. He hands you the drugs with a smirk.`
				goto greenoptions

			label end
			action
				clear "med1"
				clear "med2"
				clear "suit1"
				clear "suit2"
				clear "id1"
				clear "id2"
			`	It's never wise to linger in a place like this, especially aimlessly. With your business concluded you leave Greenrock's stalls behind.`
				decline

mission "Timothy Radrickson 5e: Back to Oblivion"
	source Oblivion
	to offer
		has "Timothy Radrickson 5a: Timshank Redemption: done"
	on offer

> 		"timothy radrickson preparedness" = "timothy radrickson	medicine" + "timothy radrickson suit" + "timothy radrickson id"

Reviewer 9: Oddly enough this doesn't work in-game if it's not alphabetical

![image](https://github.com/user-attachments/assets/f3ea0742-8225-451d-a8b7-e7c753753fda)

```suggestion
		"timothy radrickson zpreparedness" = "timothy radrickson	medicine" + "timothy radrickson suit" + "timothy radrickson id"
```

Reviewer 3: I'll take your word for it; I haven't heard of this issue before.

		conversation
			`Your ship touches down beneath Oblivion's toxic ammonia-blue sky. Night or day, there is a constant flurry of workers loading, unloading, building, and expanding the ever growing boom-town of Tukayyid. You unload all of Timothy's gear onto a dolly and push it out into the busy street.`
			`	Your presence goes unnoticed among the general ruckus and you're able to slip into the off-loading warehouses unseen. You find Timothy there, leaning over a pile of hose hookups, breathing heavily. His blood-red eyes rise to meet yours, and despite their milky gloss, a glimmer of hope burns within them.`
			`	"You did it?" His words more strained than the last time you spoke. "You got the gear?"`
			`	He looks down at your dolly expectantly.`
			choice
				`	"It's all here."`
				`	"Of course I did."`
				`	"Yep, but you owe me big."`

			`	He nearly drops to his knees and starts sorting through the gear. He takes heart when you tell him about Wynnafirth's gun and bomb, and how highly she still thinks of him.`
			`	"Good secretary. Better friend." He wheezes as he checks out the rest of your gear.`
			branch unprepared

> 				"timothy radrickson preparedness" < 6
> 
> 			branch "decently prepared"
> 				"timothy radrickson preparedness" >= 6
> 				"timothy radrickson preparedness" <= 7
> 
> 			branch "well prepared"
> 				"timothy radrickson preparedness" > 7

Reviewer 9: ```suggestion
				"timothy radrickson zpreparedness" < 6

			branch "decently prepared"
				"timothy radrickson zpreparedness" >= 6
				"timothy radrickson zpreparedness" <= 7

			branch "well prepared"
				"timothy radrickson zpreparedness" > 7
```

			`	"Captain," he says in way of a farewell, and then he climbs aboard the truck and drives off.`

mission "Timothy Radrickson 6: News of Death"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	to offer
		has "Timothy Radrickson 5e: Back to Oblivion: offered"
		random < 5
	on offer
		event "timothy radrickson forever"
		conversation
			`A spaceport is a symphony of smells, sounds, and images in constant motion. Somewhere in the din, a news report is playing, and a familiar name leaps out at you. Your attention piqued, you turn to watch.`
			`	"News out of Oblivion today. A minor explosion has the claimed the life of Timothy Radrickson, former disgraced Governor of Rand. Sentenced to prison-labor on Oblivion, a hard but fair justice for a man who..."`
			`	The news broadcast continues to blare, but you aren't able to follow any more than that.`
				decline

event "timothy radrickson forever"

mission "Timothy Radrickson 7: Monk"
	source "New Tibet"
	to offer
		random < 25
		has "event: timothy radrickson forever"

> 		"timothy radrickson preparedness" > 7

Reviewer 9: ```suggestion
		"timothy radrickson zpreparedness" > 7
```

				decline
		
			label offer
			action
				set "offers"
			`	Timothy's eyes go wide. "No, you couldn't. It's yours. You've given me too much already."`
			choice
				`	(Tell him about what Wynnafrith said, and how he is the real Hero of Rand.)`
					goto offer2
				`	(Tell him you don't need the money, but he does.)`
					goto offer2
				`	(Agree with him and rescind your offer.)`
					goto goodbyes
			
			label offer2
			action
				clear "salary: Hero of Rand"
				clear "offers"
			`	As you speak Timothy shakes his head slowly, tears streaming down his face. Finally, as you finish, he wraps his arms around you in an earnest, heartfelt hug.`
			`	He tries to speak, but his voice fails him. Instead he just shakes his teary-eyed head at you and gives you a deep solemn bow.`
			`	Timothy stays to watch as you begin your slow descent down the mountain, and back to civilization.`
			decline

mission "Timothy Radrickson 7: Hospice"
	source "New Tibet"
	to offer

> 		"timothy radrickson preparedness" >= 6

Reviewer 9: ```suggestion
		"timothy radrickson zpreparedness" >= 6
```

> 		"timothy radrickson preparedness" <= 7

Reviewer 9: ```suggestion
		"timothy radrickson zpreparedness" <= 7
```

			choice
				`	(Tell him about what Wynnafrith said, and how he is the real Hero of Rand.)`
					goto offer2
				`	(Tell him you don't need the money, but he does.)`
					goto offer2
				`	(Agree with him and rescind your offer.)`
					goto goodbyes
			
			label offer2
			action
				clear "salary: Hero of Rand"
				clear "offers"
			`	As you speak Timothy shakes his head slowly, tears streaming down his face. Finally, as you finish, he reaches out to you with his one good arm and pulls you in for a hug.`
			`	He tries to speak, but his voice fails him. Instead he just shakes his teary-eyed head and bows it before you in deep respect.`
			`	"Always." He manages to whisper through phlegm and weakened lungs as you turn to leave, "always you."`
			`	You leave him bedridden in the hospital, and to your surprise find your guide waiting for you outside the room.`
			`	"His is a tough case. Perhaps one day he will recover enough to leave this place." The monk pauses, thoughtfully. "Or perhaps not."`
			`	He leads you back to the train station, and from there you take the brief journey back to New Tibet's spaceport.`

> 				decline

Reviewer 9: ```suggestion
```
Otherwise
![image](https://github.com/user-attachments/assets/517f8dc0-1539-4542-aff4-97d42a98bfa1)

Reviewer 3: That's odd, but I suppose that `decline` node is unnecessary.

mission "Timothy Radrickson 7: Grave"
	source "Oblivion"
	to offer
		random < 25
		has "event: timothy radrickson forever"
		or

> 			"timothy radrickson preparedness" < 6

Reviewer 9: ```suggestion
			"timothy radrickson zpreparedness" < 6
```

			choice
				`	(Accept the money.)`
					goto moneyaccept
				`	(Refuse the money.)`

			`	You begin to refuse the generous offer, but Timothy holds up a hand to stop you. "I am doing quite well, and this discharges a debt that has been weighing on me. Besides, the money's already been sent!"`

			label moneyaccept
			`	Timothy pulls out a bottle beneath his desk and two glass cups. "To celebrate our reunion."`
			`	He pours the brown liquid in the cups and pushes one to you. "Things have gone well for me, and are only going to get better from here on out."`
			`	He takes a sip and says, "We've just launched our largest freighter fleet to date." He sits back in his desk, relaxed, and gets a dreamy look in his eyes. "When it returns from Wayfarer, the government of Rand will finally have enough credits to make some real changes around here. Give something back to the working man."`
			choice
				`	(Congratulate Timothy.)`
				`	(Take a sip from your glass.)`
				`	(Complain about the living standards of workers on Rand.)`

> 				`	(Complain about hand-outs for the poor of Rand.)`

Reviewer 3: Potentially I'd write what the captain would actually say (in places where they are given the option to say something) instead of just telling the player they said something; it's not as good this way, especially if you read the mission conversation like a book. Wonder why I did not do this earlier.

Author: Done.

			`	"We thought you're gonna bail out on us for bein' late. Thankfully you got the sense not to. And now ya get the goods in exchange." The middle kid hands you the ID. "Good dealin' with ya," he says as they walk off.`
			`	As you go back to the spaceport, you take a look at the ID for the first time.`
			`	Whoever made this clearly wasn't a professional. It looks like they've somehow botched the bar code, given the smear on the print, and the duplication of the emblem of the planetary authority of New Britain on the back is rather dodgy. But unless you can find a better alternative, it'll have to do.`
				goto options

			label leaveid
			`	You sigh, having learnt a lesson that teenagers in parking lots probably aren't the most trustworthy of people. Back to the spaceport.`
				goto options

			label end
			action
				clear "med1"
				clear "med2"
				clear "suit1"
				clear "suit2"
				clear "id1"
			`	After a long day of walking about the New British spaceport and wading through the crowds of hustlers and travellers, you return to the relatively calmer terminal area at the top of the pyramid.`
				decline

> 

Reviewer 9: These missions probably shouldn't continue indefinitely once you've finished or failed to save Tim since you only need to complete one or two of them. I _think_ the below should do it
```suggestion
	to fail
		or
			has "Timothy Radrickson 5a: Timshank Redemption: done"
			has "Timothy Radrickson 5a: Timshank Redemption: failed"

```

				goto danceroptions

			label id2
			action
				set "id2"
			`	You return to the bin with Skounce Yarling's ID sitting on the top of his forlorn refuse.`
			choice
				`	(Buy the ID for Tim.)`
					goto getid
				`	(Pass.)`
			`	You let sleeping IDs lie, and return to the rest of the store.`
				goto danceroptions

			label getid
			action
				set "id2"
				"timothy radrickson id" = 2
				payment -1000
			`	Bertha favors you with a mischievous grin and combs her black hair behind her ears. "I must admit, I have a thing for danger." Her eyes sparkle and her voice is a little huskier than before as she says, "I'm not sure whatchya need this for, and I certainly don't want you to tell me. But I hope you pull it off."`
			`	She takes your <payment> and hands you the old ID, her eyes lingering on you.`
				goto danceroptions

			label end
			action
				clear "med1"
				clear "med2"
				clear "suit1"
				clear "suit2"
				clear "id1"
				clear "id2"
			`	Bertha gives you a friendly nod as you make your way back outside into Dancer's foreboding weather, and the rest of the spaceport.`
				decline

> 			

Reviewer 9: ```suggestion
	to fail
		or
			has "Timothy Radrickson 5a: Timshank Redemption: done"
			has "Timothy Radrickson 5a: Timshank Redemption: failed"

```

			choice
				`	(Buy the expired detoxification medicine for Tim.)`
					goto getmed
				`	(Pass.)`

			`	The drug-dealer says as you go away, "You won't find a better deal than this. I'll be here if you change your mind."`
				goto greenoptions

			label med2
			action
				set "med2"
			`	The drug-dealer's knowing smile is almost a sneer as you return to his stall. He waits to see if you're ready to buy something.`
			choice
				`	(Buy the expired detoxification medicine for Tim.)`
					goto getmed
				`	(Pass.)`

			`	You leave the drug-dealer behind and head back into the emporium.`
				goto greenoptions
			
			label getmed
			action
				set "med2"
				"timothy radrickson	medicine" = 2
				payment -50000
			`	"Trust me, it's still good," the drug-dealer says as he takes your <payment>. He hands you the drugs with a smirk.`
				goto greenoptions

			label end
			action
				clear "med1"
				clear "med2"
				clear "suit1"
				clear "suit2"
				clear "id1"
				clear "id2"
			`	It's never wise to linger in a place like this, especially aimlessly. With your business concluded you leave Greenrock's stalls behind.`
				decline

> 

Reviewer 9: ```suggestion
	to fail
		or
			has "Timothy Radrickson 5a: Timshank Redemption: done"
			has "Timothy Radrickson 5a: Timshank Redemption: failed"

```

			`	As your ship approaches the mine you can't help but notice the burnt out remains of a gutted and abandoned town. A few minutes later your ship rattles down outside the mine's entrance.`
			choice
				`	(Leave your ship.)`
			`	It's dark and dusty outside. The planet's gravity makes you feel slow, and the air uncomfortably heavy in your lungs. The entrance to the mine looms large before you and seems suddenly quite menacing. Perhaps this wasn't the best idea...`
			`	"Don't move, or we will end you!" comes a shout from within the darkness of the mine. "Identify yourself, now!"`
			choice
				`	"I'm Captain <last>, veteran of the human civil war!"`
					goto timsbud
				`	"I'm a friend of Timothy Radrickson!"`
					goto timsbud
				`	"I'm the Hero of Rand!"`

> 					goto timsbud

Reviewer 9: ```suggestion
					goto timsbud
					to display
						has "Timothy Radrickson 3d: Return to Rand: done"
```

				`	"I ran into him at the Rand spaceport."`
				`	"He sent me a message that he wanted to meet me."`
				`	"I found him in an old equipment locker."`
				`	"He was hiding in a food crate on my ship."`
					goto right

			`	You hear someone swear and say, "We gotta move now! Everything we can't carry, burn; they're onto us!" Some commotion ensues and you hear a few individuals run into the cave.`
			`	The remaining guard hears something on the radio and then says, "No loose ends. Got it."`
			choice
				`	(Run after them into the cave.)`
				`	(Run back to your ship.)`
					goto "to the ship"
			`	You run straight for the cave, hand on pistol and heart racing. Unfortunately for you, instead of the cave you run right into a bullet. The last thing you see is the guard's dirtied face looking over your body before he finishes the job.`
				die

			label "to the ship"
			`	You turn around and dash back to your ship, and find that a few bullet rounds followed you as well. You quickly take off, expecting ships to come after you; however, none do.`
				flee

			label right
			`	Despite not being able to see your potential assailants you can feel an immediate shift in the tension.`

> 			`	"Good," one of them says, seemingly more to themselves than to you. "That's real good." You hear a loud sigh and then, "Alright, Hero of Rand, you're good to approach. The boss wants to see you."`

Reviewer 9: This makes the invitation more implicit when First Last doesn't do the freighter mission, but still works (i.e. immediate shift in the tension -->gently ushered forward)
```suggestion
			`	"Good," one of them says, seemingly more to themselves than to you. "That's real good." You hear a loud sigh and then, "Alright, Hero of Rand, you're good to approach. The boss wants to see you."`
				to display
					has "Timothy Radrickson 3d: Return to Rand: done"
```
The later one on line 8267 I think still works regardless for Timothy being the real Hero of Rand, so I don't think a change is needed there

			choice
				`	"Timothy!"`
				`	"Oh, it's you."`

			`	Timothy Radrickson, former prison-worker on Oblivion, Governor of Rand, and stowaway of your ship, hugs you.`
			`	"Captain <last>. I always hoped I would see you again. Given my tenuous legal situation, I can't leave here, and I can't send any messages." He steps back to look at you.`
			`	"The years have been kind to you, Captain; you hardly look any different from when we first met! Perhaps I should have invested in ship life, instead of staying terrestrial!" He laughs at his own joke, but only modestly.`
			`	"It was difficult for me to speak when last we met on Oblivion, but you're my oldest and best friend. At my lowest points you were always there for me. I just wanted to thank you for that."`
			`	You spend the day chatting with with Timothy, before it's time to leave.`
			`	"Thank you for coming here. I wish I had something to give you, but I own nothing anymore, even though I owe you everything. Take care out there, and I'll do my best to keep out of trouble in here."`
			label goodbyes
			choice
				`	(Offer to transfer the Hero of Rand award and stipend to Timothy.)`
					goto offer
					to display

> 						not "offers"

Reviewer 9: ```suggestion
						not "offers"
						has "Timothy Radrickson 3d: Return to Rand: done"
```

			`	"The years have been kind to you, Captain; you hardly look any different from when we first met! Perhaps I should have invested in ship life, instead of staying terrestrial!" He laughs at his own joke, but only weakly given his condition. He coughs a bit, and takes a few slow deep breaths before continuing.`
			`	"It's still a little difficult for me to speak, but I feel much better than when last we met on Oblivion. I just wanted you to know you're my oldest and best friend. At my lowest points you were always there for me. So thank you for that."`
			`	You spend the day in the hospital room chatting with with Timothy, before finally having to leave.`
			`	"Thank you for coming here. I wish I had something to give you, but I own nothing anymore, even though I owe you everything. Take care out there, and I'll do my best to keep out of trouble in here."`
			label goodbyes
			choice
				`	(Offer to transfer the Hero of Rand award and stipend to Timothy.)`
					goto offer
					to display

> 						not "offers"

Reviewer 9: ```suggestion
						not "offers"
						has "Timothy Radrickson 3d: Return to Rand: done"
```

			`	"It says it expired a few years ago, but that's just so the corpos can squeeze the common man, ya know? Stuff's still good." He shrugs.`
			`	"So what'dya say? It'll cost'ya a cool 50,000 credits. But trust me, that's way cheaper than what you'll find elsewhere."`
			choice
				`	(Buy the expired detoxification medicine for Tim.)`
					goto getmed
				`	(Pass.)`

			`	The drug-dealer says as you go away, "You won't find a better deal than this. I'll be here if you change your mind."`
				goto greenoptions

			label med2
			action
				set "med2"
			`	The drug-dealer's knowing smile is almost a sneer as you return to his stall. He waits to see if you're ready to buy something.`
			choice
				`	(Buy the expired detoxification medicine for Tim.)`
					goto getmed
				`	(Pass.)`

			`	You leave the drug-dealer behind and head back into the emporium.`
				goto greenoptions
			
			label getmed
			action
				set "med2"

> 				"timothy radrickson	medicine" = 2

Reviewer 10: This seems to be a tab character rather than a space.

	to offer
		has "event: timothy radrickson: on rand"
	on offer

> 		log `Despiting starting over with nothing, Timothy the Stowaway has sent you credits to pay you back for that passage you gave him over a year ago.`

Reviewer 7: ```suggestion
		log `Despite starting over with nothing, Timothy the Stowaway has sent you credits to pay you back for that passage you gave him over a year ago.`
```

	to offer
		has "event: timothy radrickson: on rand"
	on offer

> 		log "People" "Timothy Radickson" `A stowaway I found on my ship. He requested I take him to Rand to start a new life, and I did. He promised to pay me back. And did so, sending me 50,000 credits.`

Reviewer 7: Just so you know, it appends this to what's already in the log entry. So you'll end up with duplication of information.

Author: so... should it not be removed? Or.. what?

			`	"Now!" She cuts him off, and watches as he sullenly slumps away.`
			`	Without looking your way she continues, "It was a pleasure to see you again, Captain <last>, but it's time you were leaving. The governor needs you, and my own glorious revolution needs some attention."`
			`	You leave the dilapidated mines with a bomb, a pistol, and a plan.`
			action

> 				log `I met Union-Colonel Wynnfrith, Timothy Radickson's old secretary on Rand. She appears to have become a revolutionary since I last saw her. I showed her Timothy's list, and she said I could find the survival suit on Dancer, the medication on New Britian, and the Fake ID on Shaula. She provided me with the gun and bomb Timothy asked for.`

Reviewer 5: ```suggestion
				log `Met Union-Colonel Wynnfrith, Timothy Radickson's old secretary on Rand. She appears to have become a revolutionary since I last saw her. Showed her Timothy's list, and she said I could find a survival suit on Dancer, medication on New Britain, and a fake ID on Shaula. Provided me with the gun and bomb Timothy asked for.`
```

Author: Made these changes.

		conversation
			`You feel the pressure of many unseen eyes watching you as you leave the spaceport and enter into the sprawling stalls of Greenrock's rowdy emporium, where everything and everyone is for sale.`
			`	Greenrock looks a little more civil than when you were racing around saving captured freighter fleets, but only just a little. There is a wildness here and violence simmers just beneath the surface of every interaction. Luckily, no one seems to remember or care about the fleet you destroyed.`

> 			to display

Reviewer 7: ```suggestion
				to display
```

> 				has "Timothy Radrickson 3d: Return to Rand: done"

Reviewer 7: ```suggestion
					has "Timothy Radrickson 3d: Return to Rand: done"
```
